<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 11 Speciation | A Primer of Evolution</title>
  <meta name="description" content="An Introduction to Evolutionary Thought: Theory, Evidence, and Practice" />
  <meta name="generator" content="bookdown 0.42 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 11 Speciation | A Primer of Evolution" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="An Introduction to Evolutionary Thought: Theory, Evidence, and Practice" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 11 Speciation | A Primer of Evolution" />
  
  <meta name="twitter:description" content="An Introduction to Evolutionary Thought: Theory, Evidence, and Practice" />
  

<meta name="author" content="Michi Tobler" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="social-behavior-and-sexual-selection.html"/>
<link rel="next" href="evolutionary-medicine-i-aging-and-diseases-of-civilization.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-F33BYJVTN7"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-F33BYJVTN7');
</script>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-NCY5ZWLVNZ"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-NCY5ZWLVNZ');
</script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#in-the-light-of-evolution"><i class="fa fa-check"></i>In the Light of Evolution</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#beyond-understanding-biodiversity"><i class="fa fa-check"></i>Beyond Understanding Biodiversity</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#an-overview-of-the-semester"><i class="fa fa-check"></i>An Overview of the Semester</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#part-1-the-basics"><i class="fa fa-check"></i>Part 1: The Basics</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#part-2-a-genetic-perspective-on-evolution"><i class="fa fa-check"></i>Part 2: A Genetic Perspective on Evolution</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#part-3-evolutionary-outcomes-in-the-real-world"><i class="fa fa-check"></i>Part 3: Evolutionary Outcomes in the Real World</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#part-4-applied-evolutionary-biology"><i class="fa fa-check"></i>Part 4: Applied Evolutionary Biology</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#how-to-use-this-book"><i class="fa fa-check"></i>How to Use This Book</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#references"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="front-matters.html"><a href="front-matters.html"><i class="fa fa-check"></i>Front Matters</a>
<ul>
<li class="chapter" data-level="" data-path="front-matters.html"><a href="front-matters.html#acknowledgments"><i class="fa fa-check"></i>Acknowledgments</a></li>
<li class="chapter" data-level="" data-path="front-matters.html"><a href="front-matters.html#copyright"><i class="fa fa-check"></i>Copyright</a></li>
<li class="chapter" data-level="" data-path="front-matters.html"><a href="front-matters.html#about-the-author"><i class="fa fa-check"></i>About the Author</a></li>
</ul></li>
<li class="part"><span><b>I Basic Concepts</b></span></li>
<li class="chapter" data-level="1" data-path="what-evolution-is.html"><a href="what-evolution-is.html"><i class="fa fa-check"></i><b>1</b> What Evolution Is</a>
<ul>
<li class="chapter" data-level="1.1" data-path="what-evolution-is.html"><a href="what-evolution-is.html#darwin-and-the-conception-of-a-new-idea"><i class="fa fa-check"></i><b>1.1</b> Darwin and the Conception of a New Idea</a></li>
<li class="chapter" data-level="1.2" data-path="what-evolution-is.html"><a href="what-evolution-is.html#two-fundamental-insights"><i class="fa fa-check"></i><b>1.2</b> Two Fundamental Insights</a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="what-evolution-is.html"><a href="what-evolution-is.html#predictions"><i class="fa fa-check"></i><b>1.2.1</b> The Pattern: Evolution is Descent with Modification</a></li>
<li class="chapter" data-level="1.2.2" data-path="what-evolution-is.html"><a href="what-evolution-is.html#the-process-natural-selection-is-a-mechanism-of-evolution"><i class="fa fa-check"></i><b>1.2.2</b> The Process: Natural Selection is a Mechanism of Evolution</a></li>
<li class="chapter" data-level="1.2.3" data-path="what-evolution-is.html"><a href="what-evolution-is.html#distinguishing-between-pattern-and-process"><i class="fa fa-check"></i><b>1.2.3</b> Distinguishing between Pattern and Process</a></li>
<li class="chapter" data-level="1.2.4" data-path="what-evolution-is.html"><a href="what-evolution-is.html#population-thinking"><i class="fa fa-check"></i><b>1.2.4</b> Population Thinking</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="what-evolution-is.html"><a href="what-evolution-is.html#practical-skills-getting-started-with-r"><i class="fa fa-check"></i><b>1.3</b> Practical Skills: Getting Started with R</a>
<ul>
<li class="chapter" data-level="1.3.1" data-path="what-evolution-is.html"><a href="what-evolution-is.html#what-are-r-and-rstudio"><i class="fa fa-check"></i><b>1.3.1</b> What are R and RStudio?</a></li>
<li class="chapter" data-level="1.3.2" data-path="what-evolution-is.html"><a href="what-evolution-is.html#installing-r"><i class="fa fa-check"></i><b>1.3.2</b> Installing R</a></li>
<li class="chapter" data-level="1.3.3" data-path="what-evolution-is.html"><a href="what-evolution-is.html#installing-rstudio"><i class="fa fa-check"></i><b>1.3.3</b> Installing RStudio</a></li>
<li class="chapter" data-level="1.3.4" data-path="what-evolution-is.html"><a href="what-evolution-is.html#checking-the-successful-installation"><i class="fa fa-check"></i><b>1.3.4</b> Checking the Successful Installation</a></li>
<li class="chapter" data-level="1.3.5" data-path="what-evolution-is.html"><a href="what-evolution-is.html#some-r-and-rstudio-basics"><i class="fa fa-check"></i><b>1.3.5</b> Some R and RStudio Basics</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="what-evolution-is.html"><a href="what-evolution-is.html#additional-resources"><i class="fa fa-check"></i><b>1.4</b> Additional Resources</a>
<ul>
<li class="chapter" data-level="1.4.1" data-path="what-evolution-is.html"><a href="what-evolution-is.html#alternative-evolution-textbooks"><i class="fa fa-check"></i><b>1.4.1</b> Alternative Evolution Textbooks</a></li>
<li class="chapter" data-level="1.4.2" data-path="what-evolution-is.html"><a href="what-evolution-is.html#evolution-in-the-primary-literature-and-the-news"><i class="fa fa-check"></i><b>1.4.2</b> Evolution in the Primary Literature and the News</a></li>
<li class="chapter" data-level="1.4.3" data-path="what-evolution-is.html"><a href="what-evolution-is.html#r-and-rstudio-resources"><i class="fa fa-check"></i><b>1.4.3</b> R and RStudio Resources</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="what-evolution-is.html"><a href="what-evolution-is.html#reflection-questions"><i class="fa fa-check"></i><b>1.5</b> Reflection Questions</a></li>
<li class="chapter" data-level="1.6" data-path="what-evolution-is.html"><a href="what-evolution-is.html#references-1"><i class="fa fa-check"></i><b>1.6</b> References</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="evidence-for-evolution.html"><a href="evidence-for-evolution.html"><i class="fa fa-check"></i><b>2</b> Evidence for Evolution</a>
<ul>
<li class="chapter" data-level="2.1" data-path="evidence-for-evolution.html"><a href="evidence-for-evolution.html#microevolution"><i class="fa fa-check"></i><b>2.1</b> Microevolution</a>
<ul>
<li class="chapter" data-level="2.1.1" data-path="evidence-for-evolution.html"><a href="evidence-for-evolution.html#the-case-of-threespine-stickleback"><i class="fa fa-check"></i><b>2.1.1</b> The Case of Threespine Stickleback</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="evidence-for-evolution.html"><a href="evidence-for-evolution.html#speciation"><i class="fa fa-check"></i><b>2.2</b> Speciation</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="evidence-for-evolution.html"><a href="evidence-for-evolution.html#ring-species"><i class="fa fa-check"></i><b>2.2.1</b> Ring Species</a></li>
<li class="chapter" data-level="2.2.2" data-path="evidence-for-evolution.html"><a href="evidence-for-evolution.html#catching-speciation-in-action"><i class="fa fa-check"></i><b>2.2.2</b> Catching Speciation in Action</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="evidence-for-evolution.html"><a href="evidence-for-evolution.html#macroevolution"><i class="fa fa-check"></i><b>2.3</b> Macroevolution</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="evidence-for-evolution.html"><a href="evidence-for-evolution.html#geographic-and-temporal-patterns-of-succession"><i class="fa fa-check"></i><b>2.3.1</b> Geographic and Temporal Patterns of Succession</a></li>
<li class="chapter" data-level="2.3.2" data-path="evidence-for-evolution.html"><a href="evidence-for-evolution.html#transitional-fossils"><i class="fa fa-check"></i><b>2.3.2</b> Transitional Fossils</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="evidence-for-evolution.html"><a href="evidence-for-evolution.html#common-ancestry-and-homology"><i class="fa fa-check"></i><b>2.4</b> Common Ancestry and Homology</a>
<ul>
<li class="chapter" data-level="2.4.1" data-path="evidence-for-evolution.html"><a href="evidence-for-evolution.html#shared-flaws"><i class="fa fa-check"></i><b>2.4.1</b> Shared Flaws</a></li>
<li class="chapter" data-level="2.4.2" data-path="evidence-for-evolution.html"><a href="evidence-for-evolution.html#why-homologies-matter"><i class="fa fa-check"></i><b>2.4.2</b> Why Homologies Matter</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="evidence-for-evolution.html"><a href="evidence-for-evolution.html#deep-time"><i class="fa fa-check"></i><b>2.5</b> Deep Time</a></li>
<li class="chapter" data-level="2.6" data-path="evidence-for-evolution.html"><a href="evidence-for-evolution.html#tiktaalik"><i class="fa fa-check"></i><b>2.6</b> Correspondence of Different Lines of Evidence</a></li>
<li class="chapter" data-level="2.7" data-path="evidence-for-evolution.html"><a href="evidence-for-evolution.html#absence-of-evidence"><i class="fa fa-check"></i><b>2.7</b> Absence of Evidence…</a></li>
<li class="chapter" data-level="2.8" data-path="evidence-for-evolution.html"><a href="evidence-for-evolution.html#case-study-darwins-finches"><i class="fa fa-check"></i><b>2.8</b> Case Study: Darwin’s Finches</a></li>
<li class="chapter" data-level="2.9" data-path="evidence-for-evolution.html"><a href="evidence-for-evolution.html#practical-skills-r-notebooks-and-plotting-with-ggplot"><i class="fa fa-check"></i><b>2.9</b> Practical Skills: R Notebooks and Plotting with ggplot</a>
<ul>
<li class="chapter" data-level="2.9.1" data-path="evidence-for-evolution.html"><a href="evidence-for-evolution.html#r-notebooks"><i class="fa fa-check"></i><b>2.9.1</b> R Notebooks</a></li>
<li class="chapter" data-level="2.9.2" data-path="evidence-for-evolution.html"><a href="evidence-for-evolution.html#libraries"><i class="fa fa-check"></i><b>2.9.2</b> Using Libraries</a></li>
<li class="chapter" data-level="2.9.3" data-path="evidence-for-evolution.html"><a href="evidence-for-evolution.html#import-data"><i class="fa fa-check"></i><b>2.9.3</b> Importing Data</a></li>
<li class="chapter" data-level="2.9.4" data-path="evidence-for-evolution.html"><a href="evidence-for-evolution.html#graphing-data"><i class="fa fa-check"></i><b>2.9.4</b> Graphing Data</a></li>
</ul></li>
<li class="chapter" data-level="2.10" data-path="evidence-for-evolution.html"><a href="evidence-for-evolution.html#reflection-questions-1"><i class="fa fa-check"></i><b>2.10</b> Reflection Questions</a></li>
<li class="chapter" data-level="2.11" data-path="evidence-for-evolution.html"><a href="evidence-for-evolution.html#references-2"><i class="fa fa-check"></i><b>2.11</b> References</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="a-mechanism-for-change.html"><a href="a-mechanism-for-change.html"><i class="fa fa-check"></i><b>3</b> A Mechanism for Change</a>
<ul>
<li class="chapter" data-level="3.1" data-path="a-mechanism-for-change.html"><a href="a-mechanism-for-change.html#darwins-logic"><i class="fa fa-check"></i><b>3.1</b> Darwin’s Logic</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="a-mechanism-for-change.html"><a href="a-mechanism-for-change.html#variation"><i class="fa fa-check"></i><b>3.1.1</b> Variation</a></li>
<li class="chapter" data-level="3.1.2" data-path="a-mechanism-for-change.html"><a href="a-mechanism-for-change.html#heritability"><i class="fa fa-check"></i><b>3.1.2</b> Heritability</a></li>
<li class="chapter" data-level="3.1.3" data-path="a-mechanism-for-change.html"><a href="a-mechanism-for-change.html#the-struggle-for-existence"><i class="fa fa-check"></i><b>3.1.3</b> The Struggle for Existence</a></li>
<li class="chapter" data-level="3.1.4" data-path="a-mechanism-for-change.html"><a href="a-mechanism-for-change.html#non-random-survival-and-reproduction"><i class="fa fa-check"></i><b>3.1.4</b> Non-Random Survival and Reproduction</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="a-mechanism-for-change.html"><a href="a-mechanism-for-change.html#putting-darwins-logic-to-the-test"><i class="fa fa-check"></i><b>3.2</b> Putting Darwin’s Logic to the Test</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="a-mechanism-for-change.html"><a href="a-mechanism-for-change.html#is-there-variation-in-beak-size"><i class="fa fa-check"></i><b>3.2.1</b> Is there Variation in Beak Size?</a></li>
<li class="chapter" data-level="3.2.2" data-path="a-mechanism-for-change.html"><a href="a-mechanism-for-change.html#is-variation-in-beak-size-heritable"><i class="fa fa-check"></i><b>3.2.2</b> Is Variation in Beak Size Heritable?</a></li>
<li class="chapter" data-level="3.2.3" data-path="a-mechanism-for-change.html"><a href="a-mechanism-for-change.html#is-there-a-struggle-for-existence"><i class="fa fa-check"></i><b>3.2.3</b> Is there a Struggle for Existence?</a></li>
<li class="chapter" data-level="3.2.4" data-path="a-mechanism-for-change.html"><a href="a-mechanism-for-change.html#is-survival-non-random"><i class="fa fa-check"></i><b>3.2.4</b> Is Survival Non-Random?</a></li>
<li class="chapter" data-level="3.2.5" data-path="a-mechanism-for-change.html"><a href="a-mechanism-for-change.html#did-non-random-survival-lead-to-evolution"><i class="fa fa-check"></i><b>3.2.5</b> Did Non-Random Survival Lead to Evolution?</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="a-mechanism-for-change.html"><a href="a-mechanism-for-change.html#case-study-exploring-among-population-variation"><i class="fa fa-check"></i><b>3.3</b> Case Study: Exploring Among-Population Variation</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="a-mechanism-for-change.html"><a href="a-mechanism-for-change.html#the-cave-molly-and-its-ancestors"><i class="fa fa-check"></i><b>3.3.1</b> The Cave Molly and its Ancestors</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="a-mechanism-for-change.html"><a href="a-mechanism-for-change.html#practical-skills-number-sequences-allometry-and-histograms"><i class="fa fa-check"></i><b>3.4</b> Practical Skills: Number Sequences, Allometry, and Histograms</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="a-mechanism-for-change.html"><a href="a-mechanism-for-change.html#number-sequences"><i class="fa fa-check"></i><b>3.4.1</b> Number Sequences</a></li>
<li class="chapter" data-level="3.4.2" data-path="a-mechanism-for-change.html"><a href="a-mechanism-for-change.html#allometry-residuals"><i class="fa fa-check"></i><b>3.4.2</b> Allometry &amp; Residuals</a></li>
<li class="chapter" data-level="3.4.3" data-path="a-mechanism-for-change.html"><a href="a-mechanism-for-change.html#histograms-and-density-functions"><i class="fa fa-check"></i><b>3.4.3</b> Histograms and Density Functions</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="a-mechanism-for-change.html"><a href="a-mechanism-for-change.html#reflection-questions-2"><i class="fa fa-check"></i><b>3.5</b> Reflection Questions</a></li>
<li class="chapter" data-level="3.6" data-path="a-mechanism-for-change.html"><a href="a-mechanism-for-change.html#references-3"><i class="fa fa-check"></i><b>3.6</b> References</a></li>
</ul></li>
<li class="part"><span><b>II A Genetic Perspective on Evolution</b></span></li>
<li class="chapter" data-level="4" data-path="the-raw-materials-for-evolution.html"><a href="the-raw-materials-for-evolution.html"><i class="fa fa-check"></i><b>4</b> The Raw Materials for Evolution</a>
<ul>
<li class="chapter" data-level="4.1" data-path="the-raw-materials-for-evolution.html"><a href="the-raw-materials-for-evolution.html#the-genotype-phenotype-gap"><i class="fa fa-check"></i><b>4.1</b> The Genotype-Phenotype Gap</a></li>
<li class="chapter" data-level="4.2" data-path="the-raw-materials-for-evolution.html"><a href="the-raw-materials-for-evolution.html#mutation-the-fundamental-source-of-novelty"><i class="fa fa-check"></i><b>4.2</b> Mutation: The Fundamental Source of Novelty</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="the-raw-materials-for-evolution.html"><a href="the-raw-materials-for-evolution.html#point-mutations"><i class="fa fa-check"></i><b>4.2.1</b> Point Mutations</a></li>
<li class="chapter" data-level="4.2.2" data-path="the-raw-materials-for-evolution.html"><a href="the-raw-materials-for-evolution.html#inversions"><i class="fa fa-check"></i><b>4.2.2</b> Inversions</a></li>
<li class="chapter" data-level="4.2.3" data-path="the-raw-materials-for-evolution.html"><a href="the-raw-materials-for-evolution.html#gene-duplication"><i class="fa fa-check"></i><b>4.2.3</b> Gene Duplication</a></li>
<li class="chapter" data-level="4.2.4" data-path="the-raw-materials-for-evolution.html"><a href="the-raw-materials-for-evolution.html#genome-duplication"><i class="fa fa-check"></i><b>4.2.4</b> Genome Duplication</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="the-raw-materials-for-evolution.html"><a href="the-raw-materials-for-evolution.html#mutation-rates"><i class="fa fa-check"></i><b>4.3</b> Mutation Rates</a></li>
<li class="chapter" data-level="4.4" data-path="the-raw-materials-for-evolution.html"><a href="the-raw-materials-for-evolution.html#quantifying-genetic-variation"><i class="fa fa-check"></i><b>4.4</b> Quantifying Genetic Variation</a></li>
<li class="chapter" data-level="4.5" data-path="the-raw-materials-for-evolution.html"><a href="the-raw-materials-for-evolution.html#a-null-model-for-evolution"><i class="fa fa-check"></i><b>4.5</b> A Null Model for Evolution</a></li>
<li class="chapter" data-level="4.6" data-path="the-raw-materials-for-evolution.html"><a href="the-raw-materials-for-evolution.html#case-study-hardy-weinberg-equilirbium"><i class="fa fa-check"></i><b>4.6</b> Case Study: Hardy-Weinberg Equilirbium</a></li>
<li class="chapter" data-level="4.7" data-path="the-raw-materials-for-evolution.html"><a href="the-raw-materials-for-evolution.html#practical-skills-number-sequences-2.0-and-complex-graphs"><i class="fa fa-check"></i><b>4.7</b> Practical Skills: Number Sequences 2.0 and Complex Graphs</a>
<ul>
<li class="chapter" data-level="4.7.1" data-path="the-raw-materials-for-evolution.html"><a href="the-raw-materials-for-evolution.html#number-sequences-1"><i class="fa fa-check"></i><b>4.7.1</b> Number Sequences</a></li>
<li class="chapter" data-level="4.7.2" data-path="the-raw-materials-for-evolution.html"><a href="the-raw-materials-for-evolution.html#layering-data-sets"><i class="fa fa-check"></i><b>4.7.2</b> Layering Data Sets</a></li>
</ul></li>
<li class="chapter" data-level="4.8" data-path="the-raw-materials-for-evolution.html"><a href="the-raw-materials-for-evolution.html#reflection-questions-3"><i class="fa fa-check"></i><b>4.8</b> Reflection Questions</a></li>
<li class="chapter" data-level="4.9" data-path="the-raw-materials-for-evolution.html"><a href="the-raw-materials-for-evolution.html#references-4"><i class="fa fa-check"></i><b>4.9</b> References</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="evolutionary-mechanisms-i-modeling-selection.html"><a href="evolutionary-mechanisms-i-modeling-selection.html"><i class="fa fa-check"></i><b>5</b> Evolutionary Mechanisms I: Modeling Selection</a>
<ul>
<li class="chapter" data-level="5.1" data-path="evolutionary-mechanisms-i-modeling-selection.html"><a href="evolutionary-mechanisms-i-modeling-selection.html#the-effects-of-selection"><i class="fa fa-check"></i><b>5.1</b> The Effects of Selection</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="evolutionary-mechanisms-i-modeling-selection.html"><a href="evolutionary-mechanisms-i-modeling-selection.html#frequency-dependent-selection"><i class="fa fa-check"></i><b>5.1.1</b> Frequency-Dependent Selection</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="evolutionary-mechanisms-i-modeling-selection.html"><a href="evolutionary-mechanisms-i-modeling-selection.html#modeling-the-effects-of-selection"><i class="fa fa-check"></i><b>5.2</b> Modeling the Effects of Selection</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="evolutionary-mechanisms-i-modeling-selection.html"><a href="evolutionary-mechanisms-i-modeling-selection.html#relative-fitness"><i class="fa fa-check"></i><b>5.2.1</b> Relative Fitness</a></li>
<li class="chapter" data-level="5.2.2" data-path="evolutionary-mechanisms-i-modeling-selection.html"><a href="evolutionary-mechanisms-i-modeling-selection.html#calculating-allele-frequency-changes"><i class="fa fa-check"></i><b>5.2.2</b> Calculating Allele Frequency Changes</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="evolutionary-mechanisms-i-modeling-selection.html"><a href="evolutionary-mechanisms-i-modeling-selection.html#case-study-modeling-selection"><i class="fa fa-check"></i><b>5.3</b> Case Study: Modeling Selection</a></li>
<li class="chapter" data-level="5.4" data-path="evolutionary-mechanisms-i-modeling-selection.html"><a href="evolutionary-mechanisms-i-modeling-selection.html#practical-skills"><i class="fa fa-check"></i><b>5.4</b> Practical Skills: Modeling Selection</a>
<ul>
<li class="chapter" data-level="5.4.1" data-path="evolutionary-mechanisms-i-modeling-selection.html"><a href="evolutionary-mechanisms-i-modeling-selection.html#modeling-selection"><i class="fa fa-check"></i><b>5.4.1</b> Modeling Selection</a></li>
<li class="chapter" data-level="5.4.2" data-path="evolutionary-mechanisms-i-modeling-selection.html"><a href="evolutionary-mechanisms-i-modeling-selection.html#running-multiple-selection-models"><i class="fa fa-check"></i><b>5.4.2</b> Running Multiple Selection Models</a></li>
<li class="chapter" data-level="5.4.3" data-path="evolutionary-mechanisms-i-modeling-selection.html"><a href="evolutionary-mechanisms-i-modeling-selection.html#frequency-dependent-selection-1"><i class="fa fa-check"></i><b>5.4.3</b> Frequency-Dependent Selection</a></li>
<li class="chapter" data-level="5.4.4" data-path="evolutionary-mechanisms-i-modeling-selection.html"><a href="evolutionary-mechanisms-i-modeling-selection.html#understanding-equilibria"><i class="fa fa-check"></i><b>5.4.4</b> Understanding Equilibria</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="evolutionary-mechanisms-i-modeling-selection.html"><a href="evolutionary-mechanisms-i-modeling-selection.html#reflection-questions-4"><i class="fa fa-check"></i><b>5.5</b> Reflection Questions</a></li>
<li class="chapter" data-level="5.6" data-path="evolutionary-mechanisms-i-modeling-selection.html"><a href="evolutionary-mechanisms-i-modeling-selection.html#references-5"><i class="fa fa-check"></i><b>5.6</b> References</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="evolutionary-mechanisms-ii-mutation-genetic-drift-migration-and-non-random-mating.html"><a href="evolutionary-mechanisms-ii-mutation-genetic-drift-migration-and-non-random-mating.html"><i class="fa fa-check"></i><b>6</b> Evolutionary Mechanisms II: Mutation, Genetic Drift, Migration, and Non-Random Mating</a>
<ul>
<li class="chapter" data-level="6.1" data-path="evolutionary-mechanisms-ii-mutation-genetic-drift-migration-and-non-random-mating.html"><a href="evolutionary-mechanisms-ii-mutation-genetic-drift-migration-and-non-random-mating.html#mutation-the-force-creating-novelty"><i class="fa fa-check"></i><b>6.1</b> Mutation: The Force Creating Novelty</a></li>
<li class="chapter" data-level="6.2" data-path="evolutionary-mechanisms-ii-mutation-genetic-drift-migration-and-non-random-mating.html"><a href="evolutionary-mechanisms-ii-mutation-genetic-drift-migration-and-non-random-mating.html#genetic-drift-the-random-force"><i class="fa fa-check"></i><b>6.2</b> Genetic Drift: The Random Force</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="evolutionary-mechanisms-ii-mutation-genetic-drift-migration-and-non-random-mating.html"><a href="evolutionary-mechanisms-ii-mutation-genetic-drift-migration-and-non-random-mating.html#interactions-between-drift-and-selection"><i class="fa fa-check"></i><b>6.2.1</b> Interactions Between Drift and Selection</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="evolutionary-mechanisms-ii-mutation-genetic-drift-migration-and-non-random-mating.html"><a href="evolutionary-mechanisms-ii-mutation-genetic-drift-migration-and-non-random-mating.html#migration-the-homogenizing-force"><i class="fa fa-check"></i><b>6.3</b> Migration: The Homogenizing Force</a>
<ul>
<li class="chapter" data-level="6.3.1" data-path="evolutionary-mechanisms-ii-mutation-genetic-drift-migration-and-non-random-mating.html"><a href="evolutionary-mechanisms-ii-mutation-genetic-drift-migration-and-non-random-mating.html#interactions-between-migration-and-selection"><i class="fa fa-check"></i><b>6.3.1</b> Interactions Between Migration and Selection</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="evolutionary-mechanisms-ii-mutation-genetic-drift-migration-and-non-random-mating.html"><a href="evolutionary-mechanisms-ii-mutation-genetic-drift-migration-and-non-random-mating.html#non-random-mating-not-much-of-a-force"><i class="fa fa-check"></i><b>6.4</b> Non-Random Mating: Not Much of a Force</a>
<ul>
<li class="chapter" data-level="6.4.1" data-path="evolutionary-mechanisms-ii-mutation-genetic-drift-migration-and-non-random-mating.html"><a href="evolutionary-mechanisms-ii-mutation-genetic-drift-migration-and-non-random-mating.html#interactions-between-inbreeding-and-selection"><i class="fa fa-check"></i><b>6.4.1</b> Interactions between Inbreeding and Selection</a></li>
</ul></li>
<li class="chapter" data-level="6.5" data-path="evolutionary-mechanisms-ii-mutation-genetic-drift-migration-and-non-random-mating.html"><a href="evolutionary-mechanisms-ii-mutation-genetic-drift-migration-and-non-random-mating.html#case-study-background-beyond-selection"><i class="fa fa-check"></i><b>6.5</b> Case Study Background: Beyond Selection</a></li>
<li class="chapter" data-level="6.6" data-path="evolutionary-mechanisms-ii-mutation-genetic-drift-migration-and-non-random-mating.html"><a href="evolutionary-mechanisms-ii-mutation-genetic-drift-migration-and-non-random-mating.html#practical-skills-modeling-mutation-drift-and-selection"><i class="fa fa-check"></i><b>6.6</b> Practical Skills: Modeling Mutation, Drift, and Selection</a>
<ul>
<li class="chapter" data-level="6.6.1" data-path="evolutionary-mechanisms-ii-mutation-genetic-drift-migration-and-non-random-mating.html"><a href="evolutionary-mechanisms-ii-mutation-genetic-drift-migration-and-non-random-mating.html#modeling-mutation-and-selection"><i class="fa fa-check"></i><b>6.6.1</b> Modeling Mutation and Selection</a></li>
<li class="chapter" data-level="6.6.2" data-path="evolutionary-mechanisms-ii-mutation-genetic-drift-migration-and-non-random-mating.html"><a href="evolutionary-mechanisms-ii-mutation-genetic-drift-migration-and-non-random-mating.html#modeling-genetic-drift"><i class="fa fa-check"></i><b>6.6.2</b> Modeling Genetic Drift</a></li>
<li class="chapter" data-level="6.6.3" data-path="evolutionary-mechanisms-ii-mutation-genetic-drift-migration-and-non-random-mating.html"><a href="evolutionary-mechanisms-ii-mutation-genetic-drift-migration-and-non-random-mating.html#modeling-genetic-drift-and-selection"><i class="fa fa-check"></i><b>6.6.3</b> Modeling Genetic Drift and Selection</a></li>
</ul></li>
<li class="chapter" data-level="6.7" data-path="evolutionary-mechanisms-ii-mutation-genetic-drift-migration-and-non-random-mating.html"><a href="evolutionary-mechanisms-ii-mutation-genetic-drift-migration-and-non-random-mating.html#reflection-questions-5"><i class="fa fa-check"></i><b>6.7</b> Reflection Questions</a></li>
<li class="chapter" data-level="6.8" data-path="evolutionary-mechanisms-ii-mutation-genetic-drift-migration-and-non-random-mating.html"><a href="evolutionary-mechanisms-ii-mutation-genetic-drift-migration-and-non-random-mating.html#references-6"><i class="fa fa-check"></i><b>6.8</b> References</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="evolution-of-dna-sequences.html"><a href="evolution-of-dna-sequences.html"><i class="fa fa-check"></i><b>7</b> Evolution of DNA Sequences</a>
<ul>
<li class="chapter" data-level="7.1" data-path="evolution-of-dna-sequences.html"><a href="evolution-of-dna-sequences.html#dna-alignment-and-genome-assembly"><i class="fa fa-check"></i><b>7.1</b> DNA Alignment and Genome Assembly</a></li>
<li class="chapter" data-level="7.2" data-path="evolution-of-dna-sequences.html"><a href="evolution-of-dna-sequences.html#reconstructing-history-phylogenetic-trees"><i class="fa fa-check"></i><b>7.2</b> Reconstructing History: Phylogenetic Trees</a>
<ul>
<li class="chapter" data-level="7.2.1" data-path="evolution-of-dna-sequences.html"><a href="evolution-of-dna-sequences.html#molecular-phylogenies"><i class="fa fa-check"></i><b>7.2.1</b> Molecular Phylogenies</a></li>
<li class="chapter" data-level="7.2.2" data-path="evolution-of-dna-sequences.html"><a href="evolution-of-dna-sequences.html#molecular-phylogenies-in-real-life"><i class="fa fa-check"></i><b>7.2.2</b> Molecular Phylogenies in Real Life</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="evolution-of-dna-sequences.html"><a href="evolution-of-dna-sequences.html#the-neutral-theory"><i class="fa fa-check"></i><b>7.3</b> The Neutral Theory</a>
<ul>
<li class="chapter" data-level="7.3.1" data-path="evolution-of-dna-sequences.html"><a href="evolution-of-dna-sequences.html#molecular-clocks"><i class="fa fa-check"></i><b>7.3.1</b> Molecular Clocks</a></li>
<li class="chapter" data-level="7.3.2" data-path="evolution-of-dna-sequences.html"><a href="evolution-of-dna-sequences.html#detecting-signatures-of-selection"><i class="fa fa-check"></i><b>7.3.2</b> Detecting Signatures of Selection</a></li>
</ul></li>
<li class="chapter" data-level="7.4" data-path="evolution-of-dna-sequences.html"><a href="evolution-of-dna-sequences.html#case-study-sars-cov-2"><i class="fa fa-check"></i><b>7.4</b> Case Study: SARS-CoV-2</a>
<ul>
<li class="chapter" data-level="7.4.1" data-path="evolution-of-dna-sequences.html"><a href="evolution-of-dna-sequences.html#host-pathogen-coevolution"><i class="fa fa-check"></i><b>7.4.1</b> Host-Pathogen Coevolution</a></li>
<li class="chapter" data-level="7.4.2" data-path="evolution-of-dna-sequences.html"><a href="evolution-of-dna-sequences.html#the-evolution-of-the-seasonal-flu"><i class="fa fa-check"></i><b>7.4.2</b> The Evolution of the Seasonal Flu</a></li>
<li class="chapter" data-level="7.4.3" data-path="evolution-of-dna-sequences.html"><a href="evolution-of-dna-sequences.html#high-virulence-flu-strains"><i class="fa fa-check"></i><b>7.4.3</b> High-Virulence Flu Strains</a></li>
</ul></li>
<li class="chapter" data-level="7.5" data-path="evolution-of-dna-sequences.html"><a href="evolution-of-dna-sequences.html#practical-skills-working-with-phylogenetic-trees"><i class="fa fa-check"></i><b>7.5</b> Practical Skills: Working with Phylogenetic Trees</a>
<ul>
<li class="chapter" data-level="7.5.1" data-path="evolution-of-dna-sequences.html"><a href="evolution-of-dna-sequences.html#installing-dependencies"><i class="fa fa-check"></i><b>7.5.1</b> Installing Dependencies</a></li>
<li class="chapter" data-level="7.5.2" data-path="evolution-of-dna-sequences.html"><a href="evolution-of-dna-sequences.html#the-pipe-operator"><i class="fa fa-check"></i><b>7.5.2</b> The Pipe Operator</a></li>
<li class="chapter" data-level="7.5.3" data-path="evolution-of-dna-sequences.html"><a href="evolution-of-dna-sequences.html#importing-a-tree"><i class="fa fa-check"></i><b>7.5.3</b> Importing a Tree</a></li>
<li class="chapter" data-level="7.5.4" data-path="evolution-of-dna-sequences.html"><a href="evolution-of-dna-sequences.html#plotting-a-tree"><i class="fa fa-check"></i><b>7.5.4</b> Plotting a Tree</a></li>
<li class="chapter" data-level="7.5.5" data-path="evolution-of-dna-sequences.html"><a href="evolution-of-dna-sequences.html#annotating-a-tree"><i class="fa fa-check"></i><b>7.5.5</b> Annotating a Tree</a></li>
<li class="chapter" data-level="7.5.6" data-path="evolution-of-dna-sequences.html"><a href="evolution-of-dna-sequences.html#calculating-the-distance-to-root"><i class="fa fa-check"></i><b>7.5.6</b> Calculating the Distance to Root</a></li>
</ul></li>
<li class="chapter" data-level="7.6" data-path="evolution-of-dna-sequences.html"><a href="evolution-of-dna-sequences.html#reflection-questions-6"><i class="fa fa-check"></i><b>7.6</b> Reflection Questions</a></li>
<li class="chapter" data-level="7.7" data-path="evolution-of-dna-sequences.html"><a href="evolution-of-dna-sequences.html#references-7"><i class="fa fa-check"></i><b>7.7</b> References</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="the-evolution-of-quantitative-traits.html"><a href="the-evolution-of-quantitative-traits.html"><i class="fa fa-check"></i><b>8</b> The Evolution of Quantitative Traits</a>
<ul>
<li class="chapter" data-level="8.1" data-path="the-evolution-of-quantitative-traits.html"><a href="the-evolution-of-quantitative-traits.html#qualitative-traits-and-epistasis"><i class="fa fa-check"></i><b>8.1</b> Qualitative Traits and Epistasis</a></li>
<li class="chapter" data-level="8.2" data-path="the-evolution-of-quantitative-traits.html"><a href="the-evolution-of-quantitative-traits.html#quantitative-traits-a-product-of-genes-and-environment"><i class="fa fa-check"></i><b>8.2</b> Quantitative Traits: a Product of Genes and Environment</a></li>
<li class="chapter" data-level="8.3" data-path="the-evolution-of-quantitative-traits.html"><a href="the-evolution-of-quantitative-traits.html#quantitative-trait-evolution"><i class="fa fa-check"></i><b>8.3</b> Quantitative Trait Evolution</a>
<ul>
<li class="chapter" data-level="8.3.1" data-path="the-evolution-of-quantitative-traits.html"><a href="the-evolution-of-quantitative-traits.html#quantifying-trait-heritability"><i class="fa fa-check"></i><b>8.3.1</b> Quantifying Trait Heritability</a></li>
<li class="chapter" data-level="8.3.2" data-path="the-evolution-of-quantitative-traits.html"><a href="the-evolution-of-quantitative-traits.html#quantifying-the-strength-of-selection"><i class="fa fa-check"></i><b>8.3.2</b> Quantifying the Strength of Selection</a></li>
<li class="chapter" data-level="8.3.3" data-path="the-evolution-of-quantitative-traits.html"><a href="the-evolution-of-quantitative-traits.html#calculating-the-response-to-selection"><i class="fa fa-check"></i><b>8.3.3</b> Calculating the Response to Selection</a></li>
</ul></li>
<li class="chapter" data-level="8.4" data-path="the-evolution-of-quantitative-traits.html"><a href="the-evolution-of-quantitative-traits.html#modes-of-selection-on-quantitative-traits"><i class="fa fa-check"></i><b>8.4</b> Modes of Selection on Quantitative Traits</a>
<ul>
<li class="chapter" data-level="8.4.1" data-path="the-evolution-of-quantitative-traits.html"><a href="the-evolution-of-quantitative-traits.html#directional-selection"><i class="fa fa-check"></i><b>8.4.1</b> Directional Selection</a></li>
<li class="chapter" data-level="8.4.2" data-path="the-evolution-of-quantitative-traits.html"><a href="the-evolution-of-quantitative-traits.html#stabilizing-selection"><i class="fa fa-check"></i><b>8.4.2</b> Stabilizing Selection</a></li>
<li class="chapter" data-level="8.4.3" data-path="the-evolution-of-quantitative-traits.html"><a href="the-evolution-of-quantitative-traits.html#disruptive-selection"><i class="fa fa-check"></i><b>8.4.3</b> Disruptive Selection</a></li>
</ul></li>
<li class="chapter" data-level="8.5" data-path="the-evolution-of-quantitative-traits.html"><a href="the-evolution-of-quantitative-traits.html#identifying-genes-underlying-quantitative-traits"><i class="fa fa-check"></i><b>8.5</b> Identifying Genes Underlying Quantitative Traits</a>
<ul>
<li class="chapter" data-level="8.5.1" data-path="the-evolution-of-quantitative-traits.html"><a href="the-evolution-of-quantitative-traits.html#qtl-mapping"><i class="fa fa-check"></i><b>8.5.1</b> QTL Mapping</a></li>
<li class="chapter" data-level="8.5.2" data-path="the-evolution-of-quantitative-traits.html"><a href="the-evolution-of-quantitative-traits.html#gwas"><i class="fa fa-check"></i><b>8.5.2</b> GWAS</a></li>
</ul></li>
<li class="chapter" data-level="8.6" data-path="the-evolution-of-quantitative-traits.html"><a href="the-evolution-of-quantitative-traits.html#case-study-quantitative-genetics"><i class="fa fa-check"></i><b>8.6</b> Case Study: Quantitative Genetics</a></li>
<li class="chapter" data-level="8.7" data-path="the-evolution-of-quantitative-traits.html"><a href="the-evolution-of-quantitative-traits.html#practical-skills-qtl-mapping"><i class="fa fa-check"></i><b>8.7</b> Practical Skills: QTL Mapping</a>
<ul>
<li class="chapter" data-level="8.7.1" data-path="the-evolution-of-quantitative-traits.html"><a href="the-evolution-of-quantitative-traits.html#installing-dependencies-1"><i class="fa fa-check"></i><b>8.7.1</b> Installing Dependencies</a></li>
<li class="chapter" data-level="8.7.2" data-path="the-evolution-of-quantitative-traits.html"><a href="the-evolution-of-quantitative-traits.html#read-qtl-data"><i class="fa fa-check"></i><b>8.7.2</b> Read QTL Data</a></li>
<li class="chapter" data-level="8.7.3" data-path="the-evolution-of-quantitative-traits.html"><a href="the-evolution-of-quantitative-traits.html#explore-the-data-genotypes-loci-linkage-map"><i class="fa fa-check"></i><b>8.7.3</b> Explore the Data: Genotypes, Loci &amp; Linkage Map</a></li>
<li class="chapter" data-level="8.7.4" data-path="the-evolution-of-quantitative-traits.html"><a href="the-evolution-of-quantitative-traits.html#explore-the-data-phenotypes"><i class="fa fa-check"></i><b>8.7.4</b> Explore the Data: Phenotypes</a></li>
<li class="chapter" data-level="8.7.5" data-path="the-evolution-of-quantitative-traits.html"><a href="the-evolution-of-quantitative-traits.html#qtl-scan"><i class="fa fa-check"></i><b>8.7.5</b> QTL Scan</a></li>
</ul></li>
<li class="chapter" data-level="8.8" data-path="the-evolution-of-quantitative-traits.html"><a href="the-evolution-of-quantitative-traits.html#reflection-questions-7"><i class="fa fa-check"></i><b>8.8</b> Reflection Questions</a></li>
<li class="chapter" data-level="8.9" data-path="the-evolution-of-quantitative-traits.html"><a href="the-evolution-of-quantitative-traits.html#references-8"><i class="fa fa-check"></i><b>8.9</b> References</a></li>
</ul></li>
<li class="part"><span><b>III Evolution in the Real World</b></span></li>
<li class="chapter" data-level="9" data-path="adaptation-and-phenotypic-plasticity.html"><a href="adaptation-and-phenotypic-plasticity.html"><i class="fa fa-check"></i><b>9</b> Adaptation and Phenotypic Plasticity</a>
<ul>
<li class="chapter" data-level="9.1" data-path="adaptation-and-phenotypic-plasticity.html"><a href="adaptation-and-phenotypic-plasticity.html#inferring-adaptation"><i class="fa fa-check"></i><b>9.1</b> Inferring Adaptation</a>
<ul>
<li class="chapter" data-level="9.1.1" data-path="adaptation-and-phenotypic-plasticity.html"><a href="adaptation-and-phenotypic-plasticity.html#interspecific-comparisons"><i class="fa fa-check"></i><b>9.1.1</b> Interspecific Comparisons</a></li>
<li class="chapter" data-level="9.1.2" data-path="adaptation-and-phenotypic-plasticity.html"><a href="adaptation-and-phenotypic-plasticity.html#intraspecific-studies"><i class="fa fa-check"></i><b>9.1.2</b> Intraspecific Studies</a></li>
<li class="chapter" data-level="9.1.3" data-path="adaptation-and-phenotypic-plasticity.html"><a href="adaptation-and-phenotypic-plasticity.html#local-adaptation-within-species"><i class="fa fa-check"></i><b>9.1.3</b> Local Adaptation within Species</a></li>
</ul></li>
<li class="chapter" data-level="9.2" data-path="adaptation-and-phenotypic-plasticity.html"><a href="adaptation-and-phenotypic-plasticity.html#phenotypic-plasticity"><i class="fa fa-check"></i><b>9.2</b> Phenotypic Plasticity</a>
<ul>
<li class="chapter" data-level="9.2.1" data-path="adaptation-and-phenotypic-plasticity.html"><a href="adaptation-and-phenotypic-plasticity.html#types-of-phenotypic-plasticity"><i class="fa fa-check"></i><b>9.2.1</b> Types of Phenotypic Plasticity</a></li>
<li class="chapter" data-level="9.2.2" data-path="adaptation-and-phenotypic-plasticity.html"><a href="adaptation-and-phenotypic-plasticity.html#reaction-norms-and-the-evolution-of-plasticity"><i class="fa fa-check"></i><b>9.2.2</b> Reaction Norms and the Evolution of Plasticity</a></li>
<li class="chapter" data-level="9.2.3" data-path="adaptation-and-phenotypic-plasticity.html"><a href="adaptation-and-phenotypic-plasticity.html#adaptive-and-maladaptive-phenotypic-plasticity"><i class="fa fa-check"></i><b>9.2.3</b> Adaptive and Maladaptive Phenotypic Plasticity</a></li>
</ul></li>
<li class="chapter" data-level="9.3" data-path="adaptation-and-phenotypic-plasticity.html"><a href="adaptation-and-phenotypic-plasticity.html#case-study-local-adaptation-and-phenotypic-plasticity"><i class="fa fa-check"></i><b>9.3</b> Case Study: Local Adaptation and Phenotypic Plasticity</a></li>
<li class="chapter" data-level="9.4" data-path="adaptation-and-phenotypic-plasticity.html"><a href="adaptation-and-phenotypic-plasticity.html#practical-skills-more-variables-more-graphs"><i class="fa fa-check"></i><b>9.4</b> Practical Skills: More Variables, More Graphs</a>
<ul>
<li class="chapter" data-level="9.4.1" data-path="adaptation-and-phenotypic-plasticity.html"><a href="adaptation-and-phenotypic-plasticity.html#example-1-many-independent-variables"><i class="fa fa-check"></i><b>9.4.1</b> Example 1: Many Independent Variables</a></li>
<li class="chapter" data-level="9.4.2" data-path="adaptation-and-phenotypic-plasticity.html"><a href="adaptation-and-phenotypic-plasticity.html#example-2-many-dependent-variables"><i class="fa fa-check"></i><b>9.4.2</b> Example 2: Many Dependent Variables</a></li>
</ul></li>
<li class="chapter" data-level="9.5" data-path="adaptation-and-phenotypic-plasticity.html"><a href="adaptation-and-phenotypic-plasticity.html#reflection-questions-8"><i class="fa fa-check"></i><b>9.5</b> Reflection Questions</a></li>
<li class="chapter" data-level="9.6" data-path="adaptation-and-phenotypic-plasticity.html"><a href="adaptation-and-phenotypic-plasticity.html#references-9"><i class="fa fa-check"></i><b>9.6</b> References</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="social-behavior-and-sexual-selection.html"><a href="social-behavior-and-sexual-selection.html"><i class="fa fa-check"></i><b>10</b> Social Behavior and Sexual Selection</a>
<ul>
<li class="chapter" data-level="10.1" data-path="social-behavior-and-sexual-selection.html"><a href="social-behavior-and-sexual-selection.html#social-behavior"><i class="fa fa-check"></i><b>10.1</b> Social Behavior</a>
<ul>
<li class="chapter" data-level="10.1.1" data-path="social-behavior-and-sexual-selection.html"><a href="social-behavior-and-sexual-selection.html#kin-selection"><i class="fa fa-check"></i><b>10.1.1</b> Kin Selection</a></li>
<li class="chapter" data-level="10.1.2" data-path="social-behavior-and-sexual-selection.html"><a href="social-behavior-and-sexual-selection.html#reciprocal-altruism"><i class="fa fa-check"></i><b>10.1.2</b> Reciprocal Altruism</a></li>
</ul></li>
<li class="chapter" data-level="10.2" data-path="social-behavior-and-sexual-selection.html"><a href="social-behavior-and-sexual-selection.html#sexual-selection"><i class="fa fa-check"></i><b>10.2</b> Sexual Selection</a>
<ul>
<li class="chapter" data-level="10.2.1" data-path="social-behavior-and-sexual-selection.html"><a href="social-behavior-and-sexual-selection.html#causes-of-sexual-selection"><i class="fa fa-check"></i><b>10.2.1</b> Causes of Sexual Selection</a></li>
<li class="chapter" data-level="10.2.2" data-path="social-behavior-and-sexual-selection.html"><a href="social-behavior-and-sexual-selection.html#intrasexual-selection-male-male-competition"><i class="fa fa-check"></i><b>10.2.2</b> Intrasexual Selection: Male-Male Competition</a></li>
<li class="chapter" data-level="10.2.3" data-path="social-behavior-and-sexual-selection.html"><a href="social-behavior-and-sexual-selection.html#sexual-conflict"><i class="fa fa-check"></i><b>10.2.3</b> Sexual Conflict</a></li>
<li class="chapter" data-level="10.2.4" data-path="social-behavior-and-sexual-selection.html"><a href="social-behavior-and-sexual-selection.html#intersexual-selection-female-mate-choice"><i class="fa fa-check"></i><b>10.2.4</b> Intersexual Selection: Female Mate Choice</a></li>
</ul></li>
<li class="chapter" data-level="10.3" data-path="social-behavior-and-sexual-selection.html"><a href="social-behavior-and-sexual-selection.html#case-study-altruism-and-sexual-selection"><i class="fa fa-check"></i><b>10.3</b> Case Study: Altruism and Sexual Selection</a></li>
<li class="chapter" data-level="10.4" data-path="social-behavior-and-sexual-selection.html"><a href="social-behavior-and-sexual-selection.html#reflection-questions-9"><i class="fa fa-check"></i><b>10.4</b> Reflection Questions</a></li>
<li class="chapter" data-level="10.5" data-path="social-behavior-and-sexual-selection.html"><a href="social-behavior-and-sexual-selection.html#references-10"><i class="fa fa-check"></i><b>10.5</b> References</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="speciation-1.html"><a href="speciation-1.html"><i class="fa fa-check"></i><b>11</b> Speciation</a>
<ul>
<li class="chapter" data-level="11.1" data-path="speciation-1.html"><a href="speciation-1.html#species-and-species-concepts"><i class="fa fa-check"></i><b>11.1</b> Species and Species Concepts</a>
<ul>
<li class="chapter" data-level="11.1.1" data-path="speciation-1.html"><a href="speciation-1.html#the-biological-species-concept"><i class="fa fa-check"></i><b>11.1.1</b> The Biological Species Concept</a></li>
</ul></li>
<li class="chapter" data-level="11.2" data-path="speciation-1.html"><a href="speciation-1.html#speciation-as-a-gradual-process"><i class="fa fa-check"></i><b>11.2</b> Speciation as a Gradual Process</a></li>
<li class="chapter" data-level="11.3" data-path="speciation-1.html"><a href="speciation-1.html#mechanisms-of-reproductive-isolation"><i class="fa fa-check"></i><b>11.3</b> Mechanisms of Reproductive Isolation</a>
<ul>
<li class="chapter" data-level="11.3.1" data-path="speciation-1.html"><a href="speciation-1.html#pre-mating-isolation"><i class="fa fa-check"></i><b>11.3.1</b> Pre-Mating Isolation</a></li>
<li class="chapter" data-level="11.3.2" data-path="speciation-1.html"><a href="speciation-1.html#post-mating-pre-zygotic-isolation"><i class="fa fa-check"></i><b>11.3.2</b> Post-Mating, Pre-zygotic Isolation</a></li>
<li class="chapter" data-level="11.3.3" data-path="speciation-1.html"><a href="speciation-1.html#post-zygotic-isolation"><i class="fa fa-check"></i><b>11.3.3</b> Post-Zygotic Isolation</a></li>
<li class="chapter" data-level="11.3.4" data-path="speciation-1.html"><a href="speciation-1.html#reproductive-isolation-as-a-complex-trait"><i class="fa fa-check"></i><b>11.3.4</b> Reproductive Isolation as a Complex Trait</a></li>
</ul></li>
<li class="chapter" data-level="11.4" data-path="speciation-1.html"><a href="speciation-1.html#the-evolution-of-reproductive-isolation"><i class="fa fa-check"></i><b>11.4</b> The Evolution of Reproductive Isolation</a>
<ul>
<li class="chapter" data-level="11.4.1" data-path="speciation-1.html"><a href="speciation-1.html#polyploidization-and-instantaneous-speciation"><i class="fa fa-check"></i><b>11.4.1</b> Polyploidization and Instantaneous Speciation</a></li>
<li class="chapter" data-level="11.4.2" data-path="speciation-1.html"><a href="speciation-1.html#allopatric-speciation-and-hybridization"><i class="fa fa-check"></i><b>11.4.2</b> Allopatric Speciation and Hybridization</a></li>
<li class="chapter" data-level="11.4.3" data-path="speciation-1.html"><a href="speciation-1.html#ecological-speciation"><i class="fa fa-check"></i><b>11.4.3</b> Ecological Speciation</a></li>
</ul></li>
<li class="chapter" data-level="11.5" data-path="speciation-1.html"><a href="speciation-1.html#adaptive-radiation"><i class="fa fa-check"></i><b>11.5</b> Adaptive Radiation</a>
<ul>
<li class="chapter" data-level="11.5.1" data-path="speciation-1.html"><a href="speciation-1.html#african-cichlid-fishes"><i class="fa fa-check"></i><b>11.5.1</b> African Cichlid Fishes</a></li>
</ul></li>
<li class="chapter" data-level="11.6" data-path="speciation-1.html"><a href="speciation-1.html#case-study-speciation"><i class="fa fa-check"></i><b>11.6</b> Case Study: Speciation</a></li>
<li class="chapter" data-level="11.7" data-path="speciation-1.html"><a href="speciation-1.html#practical-skills-interpreting-structure-plots"><i class="fa fa-check"></i><b>11.7</b> Practical Skills: Interpreting Structure Plots</a></li>
<li class="chapter" data-level="11.8" data-path="speciation-1.html"><a href="speciation-1.html#reflection-questions-10"><i class="fa fa-check"></i><b>11.8</b> Reflection Questions</a></li>
<li class="chapter" data-level="11.9" data-path="speciation-1.html"><a href="speciation-1.html#references-11"><i class="fa fa-check"></i><b>11.9</b> References</a></li>
</ul></li>
<li class="part"><span><b>IV Applied Evolutionary Biology</b></span></li>
<li class="chapter" data-level="12" data-path="evolutionary-medicine-i-aging-and-diseases-of-civilization.html"><a href="evolutionary-medicine-i-aging-and-diseases-of-civilization.html"><i class="fa fa-check"></i><b>12</b> Evolutionary Medicine I: Aging and Diseases of Civilization</a>
<ul>
<li class="chapter" data-level="12.1" data-path="evolutionary-medicine-i-aging-and-diseases-of-civilization.html"><a href="evolutionary-medicine-i-aging-and-diseases-of-civilization.html#darwinian-medicine"><i class="fa fa-check"></i><b>12.1</b> Darwinian Medicine</a></li>
<li class="chapter" data-level="12.2" data-path="evolutionary-medicine-i-aging-and-diseases-of-civilization.html"><a href="evolutionary-medicine-i-aging-and-diseases-of-civilization.html#diseases-of-civilization"><i class="fa fa-check"></i><b>12.2</b> Diseases of Civilization</a>
<ul>
<li class="chapter" data-level="12.2.1" data-path="evolutionary-medicine-i-aging-and-diseases-of-civilization.html"><a href="evolutionary-medicine-i-aging-and-diseases-of-civilization.html#modern-hunter-gatherers-and-urbanites"><i class="fa fa-check"></i><b>12.2.1</b> Modern Hunter-Gatherers and Urbanites</a></li>
<li class="chapter" data-level="12.2.2" data-path="evolutionary-medicine-i-aging-and-diseases-of-civilization.html"><a href="evolutionary-medicine-i-aging-and-diseases-of-civilization.html#old-adaptations-in-a-modern-world"><i class="fa fa-check"></i><b>12.2.2</b> Old Adaptations in a Modern World</a></li>
<li class="chapter" data-level="12.2.3" data-path="evolutionary-medicine-i-aging-and-diseases-of-civilization.html"><a href="evolutionary-medicine-i-aging-and-diseases-of-civilization.html#an-example-breast-cancer"><i class="fa fa-check"></i><b>12.2.3</b> An Example: Breast Cancer</a></li>
</ul></li>
<li class="chapter" data-level="12.3" data-path="evolutionary-medicine-i-aging-and-diseases-of-civilization.html"><a href="evolutionary-medicine-i-aging-and-diseases-of-civilization.html#why-we-age-and-die"><i class="fa fa-check"></i><b>12.3</b> Why We Age… and Die</a>
<ul>
<li class="chapter" data-level="12.3.1" data-path="evolutionary-medicine-i-aging-and-diseases-of-civilization.html"><a href="evolutionary-medicine-i-aging-and-diseases-of-civilization.html#rate-of-living-hypothesis"><i class="fa fa-check"></i><b>12.3.1</b> Rate of Living Hypothesis</a></li>
<li class="chapter" data-level="12.3.2" data-path="evolutionary-medicine-i-aging-and-diseases-of-civilization.html"><a href="evolutionary-medicine-i-aging-and-diseases-of-civilization.html#evolutionary-theories-of-aging"><i class="fa fa-check"></i><b>12.3.2</b> Evolutionary Theories of Aging</a></li>
</ul></li>
<li class="chapter" data-level="12.4" data-path="evolutionary-medicine-i-aging-and-diseases-of-civilization.html"><a href="evolutionary-medicine-i-aging-and-diseases-of-civilization.html#case-study-evolution-of-aging"><i class="fa fa-check"></i><b>12.4</b> Case Study: Evolution of Aging</a></li>
<li class="chapter" data-level="12.5" data-path="evolutionary-medicine-i-aging-and-diseases-of-civilization.html"><a href="evolutionary-medicine-i-aging-and-diseases-of-civilization.html#practical-skills-life-table-analysis"><i class="fa fa-check"></i><b>12.5</b> Practical Skills: Life Table Analysis</a></li>
<li class="chapter" data-level="12.6" data-path="evolutionary-medicine-i-aging-and-diseases-of-civilization.html"><a href="evolutionary-medicine-i-aging-and-diseases-of-civilization.html#reflection-questions-11"><i class="fa fa-check"></i><b>12.6</b> Reflection Questions</a></li>
<li class="chapter" data-level="12.7" data-path="evolutionary-medicine-i-aging-and-diseases-of-civilization.html"><a href="evolutionary-medicine-i-aging-and-diseases-of-civilization.html#references-12"><i class="fa fa-check"></i><b>12.7</b> References</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="evolutionary-medicine-ii-evolving-pathogens.html"><a href="evolutionary-medicine-ii-evolving-pathogens.html"><i class="fa fa-check"></i><b>13</b> Evolutionary Medicine II: Evolving Pathogens</a>
<ul>
<li class="chapter" data-level="13.1" data-path="evolutionary-medicine-ii-evolving-pathogens.html"><a href="evolutionary-medicine-ii-evolving-pathogens.html#epidemiology-and-pathogen-fitness"><i class="fa fa-check"></i><b>13.1</b> Epidemiology and Pathogen Fitness</a>
<ul>
<li class="chapter" data-level="13.1.1" data-path="evolutionary-medicine-ii-evolving-pathogens.html"><a href="evolutionary-medicine-ii-evolving-pathogens.html#a-basic-epidemiological-model"><i class="fa fa-check"></i><b>13.1.1</b> A Basic Epidemiological model</a></li>
<li class="chapter" data-level="13.1.2" data-path="evolutionary-medicine-ii-evolving-pathogens.html"><a href="evolutionary-medicine-ii-evolving-pathogens.html#r0-a-measure-of-pathogen-fitness"><i class="fa fa-check"></i><b>13.1.2</b> <em>R</em><sub>0</sub>: A Measure of Pathogen Fitness</a></li>
</ul></li>
<li class="chapter" data-level="13.2" data-path="evolutionary-medicine-ii-evolving-pathogens.html"><a href="evolutionary-medicine-ii-evolving-pathogens.html#the-evolution-of-virulence"><i class="fa fa-check"></i><b>13.2</b> The Evolution of Virulence</a>
<ul>
<li class="chapter" data-level="13.2.1" data-path="evolutionary-medicine-ii-evolving-pathogens.html"><a href="evolutionary-medicine-ii-evolving-pathogens.html#trade-offs-and-modes-of-transmission"><i class="fa fa-check"></i><b>13.2.1</b> Trade-Offs and Modes of Transmission</a></li>
<li class="chapter" data-level="13.2.2" data-path="evolutionary-medicine-ii-evolving-pathogens.html"><a href="evolutionary-medicine-ii-evolving-pathogens.html#short-sighted-and-coincidental-evolution-hypotheses"><i class="fa fa-check"></i><b>13.2.2</b> Short-Sighted and Coincidental Evolution Hypotheses</a></li>
</ul></li>
<li class="chapter" data-level="13.3" data-path="evolutionary-medicine-ii-evolving-pathogens.html"><a href="evolutionary-medicine-ii-evolving-pathogens.html#cultural-practices-disease-spread-and-evolution"><i class="fa fa-check"></i><b>13.3</b> Cultural Practices, Disease Spread, and Evolution</a>
<ul>
<li class="chapter" data-level="13.3.1" data-path="evolutionary-medicine-ii-evolving-pathogens.html"><a href="evolutionary-medicine-ii-evolving-pathogens.html#disrupting-pathogen-transmission"><i class="fa fa-check"></i><b>13.3.1</b> Disrupting Pathogen Transmission</a></li>
<li class="chapter" data-level="13.3.2" data-path="evolutionary-medicine-ii-evolving-pathogens.html"><a href="evolutionary-medicine-ii-evolving-pathogens.html#vaccination-and-herd-immunity"><i class="fa fa-check"></i><b>13.3.2</b> Vaccination and Herd Immunity</a></li>
<li class="chapter" data-level="13.3.3" data-path="evolutionary-medicine-ii-evolving-pathogens.html"><a href="evolutionary-medicine-ii-evolving-pathogens.html#pathogen-responses"><i class="fa fa-check"></i><b>13.3.3</b> Pathogen Responses</a></li>
</ul></li>
<li class="chapter" data-level="13.4" data-path="evolutionary-medicine-ii-evolving-pathogens.html"><a href="evolutionary-medicine-ii-evolving-pathogens.html#antibiotic-resistance"><i class="fa fa-check"></i><b>13.4</b> Antibiotic Resistance</a></li>
<li class="chapter" data-level="13.5" data-path="evolutionary-medicine-ii-evolving-pathogens.html"><a href="evolutionary-medicine-ii-evolving-pathogens.html#case-study-covid-19-epidemiology-and-the-cost-of-antibiotic-resistance"><i class="fa fa-check"></i><b>13.5</b> Case Study: COVID-19 Epidemiology and the Cost of Antibiotic Resistance</a></li>
<li class="chapter" data-level="13.6" data-path="evolutionary-medicine-ii-evolving-pathogens.html"><a href="evolutionary-medicine-ii-evolving-pathogens.html#practical-skills-data-wrangling-and-log-scales"><i class="fa fa-check"></i><b>13.6</b> Practical Skills: Data Wrangling and Log-Scales</a>
<ul>
<li class="chapter" data-level="13.6.1" data-path="evolutionary-medicine-ii-evolving-pathogens.html"><a href="evolutionary-medicine-ii-evolving-pathogens.html#data-wrangling"><i class="fa fa-check"></i><b>13.6.1</b> Data Wrangling</a></li>
</ul></li>
<li class="chapter" data-level="13.7" data-path="evolutionary-medicine-ii-evolving-pathogens.html"><a href="evolutionary-medicine-ii-evolving-pathogens.html#reflection-questions-12"><i class="fa fa-check"></i><b>13.7</b> Reflection Questions</a></li>
<li class="chapter" data-level="13.8" data-path="evolutionary-medicine-ii-evolving-pathogens.html"><a href="evolutionary-medicine-ii-evolving-pathogens.html#references-13"><i class="fa fa-check"></i><b>13.8</b> References</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="human-evolution.html"><a href="human-evolution.html"><i class="fa fa-check"></i><b>14</b> Human Evolution</a>
<ul>
<li class="chapter" data-level="14.1" data-path="human-evolution.html"><a href="human-evolution.html#phylogenetic-position"><i class="fa fa-check"></i><b>14.1</b> Phylogenetic Position</a></li>
<li class="chapter" data-level="14.2" data-path="human-evolution.html"><a href="human-evolution.html#what-the-fossils-say"><i class="fa fa-check"></i><b>14.2</b> What the Fossils Say</a>
<ul>
<li class="chapter" data-level="14.2.1" data-path="human-evolution.html"><a href="human-evolution.html#an-abridged-timeline"><i class="fa fa-check"></i><b>14.2.1</b> An Abridged Timeline</a></li>
<li class="chapter" data-level="14.2.2" data-path="human-evolution.html"><a href="human-evolution.html#evolutionary-trends"><i class="fa fa-check"></i><b>14.2.2</b> Evolutionary Trends</a></li>
<li class="chapter" data-level="14.2.3" data-path="human-evolution.html"><a href="human-evolution.html#a-lone-survivor-of-an-extinct-radiation"><i class="fa fa-check"></i><b>14.2.3</b> A Lone Survivor of an Extinct Radiation</a></li>
</ul></li>
<li class="chapter" data-level="14.3" data-path="human-evolution.html"><a href="human-evolution.html#modern-human-origins"><i class="fa fa-check"></i><b>14.3</b> Modern Human Origins</a>
<ul>
<li class="chapter" data-level="14.3.1" data-path="human-evolution.html"><a href="human-evolution.html#hypotheses"><i class="fa fa-check"></i><b>14.3.1</b> Hypotheses</a></li>
<li class="chapter" data-level="14.3.2" data-path="human-evolution.html"><a href="human-evolution.html#multiregional-evolution-vs.-out-of-africa"><i class="fa fa-check"></i><b>14.3.2</b> Multiregional Evolution <em>vs</em>. Out-Of-Africa</a></li>
<li class="chapter" data-level="14.3.3" data-path="human-evolution.html"><a href="human-evolution.html#out-of-africa-vs.-hybridization-and-assimilation"><i class="fa fa-check"></i><b>14.3.3</b> Out-of-Africa <em>vs</em>. Hybridization-and-Assimilation</a></li>
</ul></li>
<li class="chapter" data-level="14.4" data-path="human-evolution.html"><a href="human-evolution.html#what-makes-us-human"><i class="fa fa-check"></i><b>14.4</b> What Makes Us Human?</a></li>
<li class="chapter" data-level="14.5" data-path="human-evolution.html"><a href="human-evolution.html#case-study-modern-human-origins"><i class="fa fa-check"></i><b>14.5</b> Case Study: Modern Human Origins</a></li>
<li class="chapter" data-level="14.6" data-path="human-evolution.html"><a href="human-evolution.html#reflection-questions-13"><i class="fa fa-check"></i><b>14.6</b> Reflection Questions</a></li>
<li class="chapter" data-level="14.7" data-path="human-evolution.html"><a href="human-evolution.html#references-14"><i class="fa fa-check"></i><b>14.7</b> References</a></li>
</ul></li>
<li class="part"><span><b>V Interactive Simulations</b></span></li>
<li class="chapter" data-level="" data-path="simulating-selection.html"><a href="simulating-selection.html"><i class="fa fa-check"></i>Simulating Selection</a></li>
<li class="chapter" data-level="" data-path="simulating-frequency-dependent-selection.html"><a href="simulating-frequency-dependent-selection.html"><i class="fa fa-check"></i>Simulating Frequency-Dependent Selection</a></li>
<li class="chapter" data-level="" data-path="simulating-genetic-drift.html"><a href="simulating-genetic-drift.html"><i class="fa fa-check"></i>Simulating Genetic Drift</a></li>
<li class="chapter" data-level="" data-path="simulating-mutation-and-selection.html"><a href="simulating-mutation-and-selection.html"><i class="fa fa-check"></i>Simulating Mutation and Selection</a></li>
<li class="chapter" data-level="" data-path="simulating-genetic-drift-and-selection.html"><a href="simulating-genetic-drift-and-selection.html"><i class="fa fa-check"></i>Simulating Genetic Drift and Selection</a></li>
<li class="appendix"><span><b>Appendix I</b></span></li>
<li class="chapter" data-level="" data-path="graph-library.html"><a href="graph-library.html"><i class="fa fa-check"></i>Graph Library</a>
<ul>
<li class="chapter" data-level="8" data-path="graph-library.html"><a href="graph-library.html#sample-data"><i class="fa fa-check"></i><b>8</b> Sample Data</a></li>
<li class="chapter" data-level="9" data-path="graph-library.html"><a href="graph-library.html#loading-dependencies"><i class="fa fa-check"></i><b>9</b> Loading Dependencies</a></li>
<li class="chapter" data-level="10" data-path="graph-library.html"><a href="graph-library.html#one-variable-continuous"><i class="fa fa-check"></i><b>10</b> One Variable: Continuous</a>
<ul>
<li class="chapter" data-level="10.1" data-path="graph-library.html"><a href="graph-library.html#frequency-histogram"><i class="fa fa-check"></i><b>10.1</b> Frequency Histogram</a></li>
<li class="chapter" data-level="10.2" data-path="graph-library.html"><a href="graph-library.html#density-plot"><i class="fa fa-check"></i><b>10.2</b> Density Plot</a></li>
<li class="chapter" data-level="10.3" data-path="graph-library.html"><a href="graph-library.html#frequency-histogram-with-density"><i class="fa fa-check"></i><b>10.3</b> Frequency Histogram with Density</a></li>
<li class="chapter" data-level="10.4" data-path="graph-library.html"><a href="graph-library.html#adding-a-discrete-variable"><i class="fa fa-check"></i><b>10.4</b> Adding a Discrete Variable</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="graph-library.html"><a href="graph-library.html#two-variables-continuous-x-and-y"><i class="fa fa-check"></i><b>11</b> Two Variables: Continuous X and Y</a>
<ul>
<li class="chapter" data-level="11.1" data-path="graph-library.html"><a href="graph-library.html#scatter-plot"><i class="fa fa-check"></i><b>11.1</b> Scatter Plot</a></li>
<li class="chapter" data-level="11.2" data-path="graph-library.html"><a href="graph-library.html#regression-line"><i class="fa fa-check"></i><b>11.2</b> Regression Line</a></li>
<li class="chapter" data-level="11.3" data-path="graph-library.html"><a href="graph-library.html#scatter-plot-with-regression"><i class="fa fa-check"></i><b>11.3</b> Scatter Plot with Regression</a></li>
<li class="chapter" data-level="11.4" data-path="graph-library.html"><a href="graph-library.html#adding-discrete-variables"><i class="fa fa-check"></i><b>11.4</b> Adding Discrete Variables</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="graph-library.html"><a href="graph-library.html#two-variables-discrete-x-continuous-y"><i class="fa fa-check"></i><b>12</b> Two Variables: Discrete X, Continuous Y</a>
<ul>
<li class="chapter" data-level="12.1" data-path="graph-library.html"><a href="graph-library.html#box-plot"><i class="fa fa-check"></i><b>12.1</b> Box Plot</a></li>
<li class="chapter" data-level="12.2" data-path="graph-library.html"><a href="graph-library.html#dot-plot"><i class="fa fa-check"></i><b>12.2</b> Dot Plot</a></li>
<li class="chapter" data-level="12.3" data-path="graph-library.html"><a href="graph-library.html#violin-plot"><i class="fa fa-check"></i><b>12.3</b> Violin Plot</a></li>
<li class="chapter" data-level="12.4" data-path="graph-library.html"><a href="graph-library.html#adding-raw-data"><i class="fa fa-check"></i><b>12.4</b> Adding Raw data</a></li>
<li class="chapter" data-level="12.5" data-path="graph-library.html"><a href="graph-library.html#adding-additional-discrete-variables"><i class="fa fa-check"></i><b>12.5</b> Adding Additional Discrete Variables</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="graph-library.html"><a href="graph-library.html#aggregate-data"><i class="fa fa-check"></i><b>13</b> Aggregate Data</a>
<ul>
<li class="chapter" data-level="13.1" data-path="graph-library.html"><a href="graph-library.html#calculating-mean-and-standard-deviation"><i class="fa fa-check"></i><b>13.1</b> Calculating Mean and Standard Deviation</a></li>
<li class="chapter" data-level="13.2" data-path="graph-library.html"><a href="graph-library.html#visualizing-means-and-variation"><i class="fa fa-check"></i><b>13.2</b> Visualizing Means and Variation</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="graph-library.html"><a href="graph-library.html#combining-multiple-plots"><i class="fa fa-check"></i><b>14</b> Combining Multiple plots</a></li>
<li class="chapter" data-level="15" data-path="graph-library.html"><a href="graph-library.html#working-with-color-palettes"><i class="fa fa-check"></i><b>15</b> Working with Color Palettes</a>
<ul>
<li class="chapter" data-level="15.1" data-path="graph-library.html"><a href="graph-library.html#color-palettes-in-rcolorbrewer"><i class="fa fa-check"></i><b>15.1</b> Color Palettes in RColorBrewer</a></li>
<li class="chapter" data-level="15.2" data-path="graph-library.html"><a href="graph-library.html#combining-rcolorbrewer-palettes-with-ggplot"><i class="fa fa-check"></i><b>15.2</b> Combining RColorBrewer Palettes with ggplot</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="r-exercises.html"><a href="r-exercises.html"><i class="fa fa-check"></i>R Exercises</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">A Primer of Evolution</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="speciation-1" class="section level1 hasAnchor" number="11">
<h1><span class="header-section-number">Chapter 11</span> Speciation<a href="speciation-1.html#speciation-1" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>Our considerations thus far have dealt with the effects of different evolutionary forces on populations of the same species; we have not explored how lineages split to form new species. In this chapter, we will take a close look at the speciation processes. We will first clarify what a species actually is, and then discuss how the different evolutionary forces—mutation, selection, genetic drift, and migration—impact the emergence of reproductive isolation between diverging lineages.</p>
<div id="species-and-species-concepts" class="section level2 hasAnchor" number="11.1">
<h2><span class="header-section-number">11.1</span> Species and Species Concepts<a href="speciation-1.html#species-and-species-concepts" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Species are a basic evolutionary unit. Species typically consist of multiple populations, each connected by gene flow but independently shaped by mutation, selection, and genetic drift. Hence, species form the boundary for the spread of alleles. While this notion of a species is in principle simple, determining those boundaries is not. As a consequence, biologists have developed a wide variety of species concepts that use more straightforward criteria to determine the species status of different populations (see below). All of these concepts have different strengths, weaknesses, and applicabilities; but rather than dissecting these, I will just leave it with Darwin’s slightly facetious insight, because nothing has really changed since 1859:</p>
<blockquote>
<p>“<em>I shall not discuss the various definitions which have been given of the term species. No one definition has as yet satisfied all naturalists; yet every naturalist knows vaguely what he means when he speaks of a species.”</em></p>
<p>Charles Darwin</p>
</blockquote>
<p>I will say, however, that the biological species concept is the only species concept that recognizes and treats species as real natural entities. Classifications based on other species concepts are inevitably proxies, and as such, human constructs. This is not necessarily bad, because it provides the practicality needed for working with organisms—whether that is in biomedical research, argiculture, wildlife management, or conservation. It is just important to remember that species classifications do not always correspond precisely to actual biological species, a point we will return to later in this chapter.</p>
<p>I will briefly provide more background information on the biological species concept. As the only species concept that recognizes species as real natural entities, the biological species is critical if we want to understand the speciation process; it sets the criteria for when the process is completed and a single lineage has split into two.</p>
<div class="alert alert-block alert-info">
<p><b>Species Concepts: A Selection</b></p>
<p>A wide variety of concepts have been proposed to recognize, describe, and classify biodiversity. To learn more about the history and debates surrounding different species concepts, check out the article “<a href="https://blogs.scientificamerican.com/evo-eco-lab/species-concepts/">Species Concepts</a>” by Kevin Zelnio.</p>
<p>A selection of species concepts can be found below. Note that there have been concerted efforts in the recent past to unify different species concepts in an attempt to make species delimitation more consistent across taxonomic groups (De Queiroz 2007).</p>
<ul>
<li><p><strong>Biological Species Concept:</strong> A species is a group of actually or potentially interbreeding natural populations, which are reproductively isolated from other such groups.</p></li>
<li><p><strong>Cohesion Species Concept:</strong> A species is the most inclusive group of organisms having the potential for genetic and/or demographic exchangeability; a group of organisms having the potential for phenotypic cohesion through intrinsic mechanisms.</p></li>
<li><p><strong>Competition Species Concept:</strong> A species is the most extensive units in the natural economy such that reproductive competition occurs among their parts.</p></li>
<li><p><strong>Ecological Species Concept:</strong> A species is a set of organisms exploiting or adapted to a single niche (or adaptive zone).</p></li>
<li><p><strong>Evolutionary Species Concept:</strong> A species is a lineage (an ancestral-descendant sequence of populations) evolving separately from others and with its own unitary evolutionary roles, tendencies, and historical fate.</p></li>
<li><p><strong>Geneological Species Concept:</strong> A species is an exclusive group of organisms whose members more closely resemble one another than members of any outside group.</p></li>
<li><p><strong>Genetic Cluster Species Concept:</strong> A species is a group of individuals forming a distinct genotypic cluster without intermediates between other such clusters.</p></li>
<li><p><strong>Isolation Species Concept:</strong> A species is a system of populations, the gene exchange between which is prevented by absence of interbreeding between heterospecific organisms based on intrinsic properties.</p></li>
<li><p><strong>Morphological (or Phenetic) Species Concept:</strong> A species is a set of organisms that look similar to each other and is distinct from other sets.</p></li>
<li><p><strong>Phylogenetic Species concept:</strong> A species is the smallest monophyletic group distinguishable by shared derived (synapomorphic) characteristics.</p></li>
<li><p><strong>Recognition Species Concept:</strong> A species is is a group of individuals with a shared specific mate recognition or fertilization system.</p></li>
<li><p><strong>Typological Species Concept:</strong> A species is a group of organisms conforming to a common morphological plan, emphasizing the species as an essentially static, non-variable assemblage.</p></li>
</ul>
</div>
<div id="the-biological-species-concept" class="section level3 hasAnchor" number="11.1.1">
<h3><span class="header-section-number">11.1.1</span> The Biological Species Concept<a href="speciation-1.html#the-biological-species-concept" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>The biological species concept was first proposed by Ernst Mayr, who defined a species as a group of actually or potentially interbreeding natural populations, which are reproductively isolated from other such groups. Hence, species are defined by reproductive compatibility, which is meaningful from an evolutionary perspective because it hinges on the lack of gene flow.</p>
<p>It is important to stress some nuances in this definition. First of all, Mayr explicitly emphasized <em>natural</em> populations in his definition, because the ability of two species to hybridize—especially outside of their typical ecological context—does not negate their species status. As we shall see, interspecific matings between many species can lead to perfectly viable and fertile offspring, yet such matings rarely occur in their natural environment due to the reproductive barriers that prevent individuals from different species from meeting or recognizing each other as potential mates. Thus, evidence of two species hybridizing outside of their ecological context, especially in captivity, does not mean that such hybridization would occur naturally.</p>
<p>The term “potentially interbreeding” also creates frequent misconceptions. It does not refer to potential interbreeding in artificial contexts, but to potential interbreeding in the face of physical barriers that prevent individuals from different populations meeting one another. Mayr argued that we are unable to determine the species status of allopatric populations, where interbreeding is currently impossible due to physical separation. What would happen if a physical barrier was removed? For example, consider the large ungulates of the genus <em>Cervus</em> that we know as wapiti (elk) and that Europeans know as red deer. There are three disjunct populations: one in North America, one in Europe and western Asia, and one in East Asia (Figure <a href="speciation-1.html#fig:wapiti">11.1</a>). Would animals from these different populations interbreed if their ranges reconnected, or would they remain separate, reproductively isolated species? Obviously there is no way to test this in practice, and our inability to determine the species status of allopatric populations is in fact one of the biggest shortcomings of the biological species concept. Hence, taxonomist have resorted to alternative species concepts in the case of wapiti and red deer. For a long time, the different populations were considered the same species (<em>C. elaphus</em>) based on their morphological similarity and shared evolutionary history. However, recent genetic analyses have restricted <em>C. elpahus</em> to the European and western Asian forms, and the east Asian and North American forms are classified as <em>C. canadensis</em>. Which is right? Well, it’s simply a matter of perspective, because such classifications are human constructs. Note that even in populations with adjacent or overlapping ranges, determining species status based on the biological species concept can be costly and time consuming, which is why in most cases biological classification operationally still hinges on other concepts.</p>
<div class="figure"><span style="display:block;" id="fig:wapiti"></span>
<img src="images/Wapiti.png" alt="Distrubution of wapiti and red deer in North America and Eurasia. The species occurs in three disjunct populations. North American wapiti picture by Membeth (CC0); red deer picture by the [Wasp Factory](https://www.flickr.com/photos/thewaspfactory74/) ([CC BY-NC-SA 2.0](https://creativecommons.org/licenses/by-nc-sa/2.0/)); Asian wapiti picture by Membeth (CC0)." width="100%" />
<p class="caption">
Figure 11.1: Distrubution of wapiti and red deer in North America and Eurasia. The species occurs in three disjunct populations. North American wapiti picture by Membeth (CC0); red deer picture by the <a href="https://www.flickr.com/photos/thewaspfactory74/">Wasp Factory</a> (<a href="https://creativecommons.org/licenses/by-nc-sa/2.0/">CC BY-NC-SA 2.0</a>); Asian wapiti picture by Membeth (CC0).
</p>
</div>
<p>Finally, it is important to mention that the biological species concept does not apply in contexts where reproductive compatibility cannot be assessed or simply does not exist. For example, we cannot determine reproductive isolation for extinct species that we only know from the fossil record. This contributes to a long-standing debate about how many species of humans roamed this planet in prehistoric times. Human fossils are classified based on morphological traits, and twelve species are recognized in the genus <em>Homo</em> alone. However, we just don’t know whether the different phenotypic variants represent a single, morphologically diverse species, or whether there were in fact multiple, reproductively isolated species of <em>Homo</em> coexisting at one time. Similarly, the biological species concept does not apply to organisms that reproduce asexually, because different asexual individuals—by definition—are reproductively isolated from all other individuals. If there is no sex and recombination, the spread of alleles is strictly limited from mothers to daughters, and there is no other exchange of genetic material among individuals or populations.</p>
</div>
</div>
<div id="speciation-as-a-gradual-process" class="section level2 hasAnchor" number="11.2">
<h2><span class="header-section-number">11.2</span> Speciation as a Gradual Process<a href="speciation-1.html#speciation-as-a-gradual-process" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>If species are groups of actually or potentially interbreeding natural populations, which are reproductively isolated from other such groups, then speciation is the process by which reproductive isolation evolves between populations. The evolution of reproductive isolation is typically a gradual process, starting from a single variable population, continuing to differentiated populations that are still connected by gene flow, and finally ending in distinct and reproductively isolated species. This gradient is known as the speciation continuum (Figure <a href="speciation-1.html#fig:speccont2">11.2</a>). Note that movement along this speciation continuum can be bidirectional; lineages can split by accruing more and more reproductive isolation, but lineages can also merge together through hybridization.</p>
<p>Just like the evolution of other traits, changes in the degree of reproductive isolation tend to evolve gradually through time, and we can consequently observe populations at different stages along the speciation continuum in natural systems. Such systems with ongoing speciation are valuable study objects for examining how and why reproductive isolation evolves.</p>
<div class="figure"><span style="display:block;" id="fig:speccont2"></span>
<img src="images/speciation_continuum.png" alt="Speciation is not typically an instantaneous process. Rather, species evolve gradually along a speciation continuum that ranges from a single population, to differentiated populations connected by gene flow, to distinct and reproductively isolated populations." width="100%" />
<p class="caption">
Figure 11.2: Speciation is not typically an instantaneous process. Rather, species evolve gradually along a speciation continuum that ranges from a single population, to differentiated populations connected by gene flow, to distinct and reproductively isolated populations.
</p>
</div>
<p>Note that the gradual and dynamic nature of speciation is one reason for why taxonomic species may not align well with biological realities. Some natural populations can be strongly differentiated and diagnosable based on a number of other species concepts. At the same time, there may still be a lot of genes flow with other populations, and even slight changes in environmental conditions could counteract existing reproductive barriers and reverse the speciation process. In other words, not all aspects of nature can be neatly classified into different categories, and we have to contend with the fact that variation in nature is continuous because speciation is a gradual and not an instantaneous process.</p>
</div>
<div id="mechanisms-of-reproductive-isolation" class="section level2 hasAnchor" number="11.3">
<h2><span class="header-section-number">11.3</span> Mechanisms of Reproductive Isolation<a href="speciation-1.html#mechanisms-of-reproductive-isolation" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>If speciation is the evolution of reproductive isolation between populations, we first need to understand what mechanisms can mediate reproductive isolation. Only then can we consider how those mechanisms actually evolve. Broadly speaking, mechanisms of reproductive isolation can be categorized into three groups, depending on whether they prevent mating between individuals from different populations (pre-mating isolation), whether they prevent successful fertilization and the formation of a zygote (post-mating, pre-zygotic isolation), or whether they prevent the success of interpopulation hybrids (post-zygotic isolation). In this section, we will discuss some common mechanisms of reproductive isolation using illustrative examples from empirical studies.</p>
<div id="pre-mating-isolation" class="section level3 hasAnchor" number="11.3.1">
<h3><span class="header-section-number">11.3.1</span> Pre-Mating Isolation<a href="speciation-1.html#pre-mating-isolation" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Pre-mating isolation describes reproductive barriers that reduce the frequency or success of interpopulation mating. This can happen because members of different populations do not meet, do not recognize each other, or are not compatible with each other during mating.</p>
<div id="ecological-isolation" class="section level4 unnumbered hasAnchor">
<h4>Ecological Isolation<a href="speciation-1.html#ecological-isolation" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Ecological isolation reduces the probability that members of different populations encounter each other, for example as consequence of adaptation to different niches that causes spatial segregation of population. Such spatial segregation can occur at large spatial scales (<em>e.g.</em>, adaptation to different habitat types) or at small spatial scales (<em>e.g.</em>, adaptation to different host plants that grow in the same habitat patch). Two mechanisms can maintain spatial segregation between populations:</p>
<ol style="list-style-type: decimal">
<li><p>Spatial segregation can be a consequence of selection against migrants, where individuals that disperse across habitats have a reduced survivability, and, thus, a lower likelihood of encountering potential mates. Such selection against migrants is evident in desert spiders (<em>Agelenopsis aperta</em>) that have adapted to different habitat types. Woodland spiders are limited by web-sites and prey, and are selected for bold and aggressive behaviors toward prey and conspecifics alike. In contrast, riparian spider populations are released from competition but suffer from increased predation by birds, favoring more shy and reclusive behaviors (Riechert &amp; Hall 2000). Experimental translocation of spiders between woodland and riparian habitats have indicated selection against migrants, as individuals moved across habitats had significantly lower survival rates than individuals translocated within habitat types (Figure <a href="speciation-1.html#fig:habiso">11.3</a>A). Hence, any spiders that would naturally move across habitat boundaries would be selected against, reducing the likelhood of cross-population mating.</p></li>
<li><p>Spatial segregation can also be a consequence of differential habitat preferences. When individuals from locally adapted populations choose habitats based on suitability, interpopulation encounter rates can be low even in absence of strong selection against migrants. For example, stickleback fish (<em>Gasterosteus aculeatus</em>) locally adapted to either lake or stream habitats have a strong affinity to their particular habitat type (Bolnick et al. 2009). Over 80 % of marked individuals that were released at the confluence of a stream and a lake were eventually recaptured in their habitat of origin (Figure <a href="speciation-1.html#fig:habiso">11.3</a>B). They segregated spatially simply because of divergent behavioral preferences.</p></li>
</ol>
<div class="figure"><span style="display:block;" id="fig:habiso"></span>
<img src="Primer2Evolution_files/figure-html/habiso-1.png" alt="A. Translocation experiments in desert spiders (*Agelenopsis aperta*) across habitats reveals selection again migrants; translocated spiders exhibit significantly lower survival than residents in each habitat. [Data](data/11_habitat-iso2.csv) from Riechert and Hall (2000). B. Sticklebacks (*Gasterosteus aculeatus*) adapted to lake and stream habitats exhibit a strong behavioral preference for their habitat of origin. [Data](data/11_habitat-iso1.csv) from Bolnick et al. (2009)." width="100%" />
<p class="caption">
Figure 11.3: A. Translocation experiments in desert spiders (<em>Agelenopsis aperta</em>) across habitats reveals selection again migrants; translocated spiders exhibit significantly lower survival than residents in each habitat. <a href="data/11_habitat-iso2.csv">Data</a> from Riechert and Hall (2000). B. Sticklebacks (<em>Gasterosteus aculeatus</em>) adapted to lake and stream habitats exhibit a strong behavioral preference for their habitat of origin. <a href="data/11_habitat-iso1.csv">Data</a> from Bolnick et al. (2009).
</p>
</div>
<p>Segregation of populations can also occur temporally, when members of different populations are (reproductively) active during different times of the day or year. Evidence for temporal isolation comes from many flowering plants, insects, and amphibians. For example, three species of frog in the <em>Rana pipiens</em> complex have largely allopatric distributions, but their ranges overlap in parts of Texas and Oklahoma. In allopatric populations, the breeding seasons of all three species also overlap: all three species have coincident peaks in their breeding activity in the spring, and two species (<em>R. sphenocephala</em> and R. <em>berlandieri</em>) again in the fall (Figure <a href="speciation-1.html#fig:tempiso">11.4</a>). However, at sites where the three species occur sympatrically, each species has evolved a unique temporal niche, and peak breeding activities do not overlap: <em>R. sphenocephala</em> breeds in early spring, <em>R. blairi</em> in late spring, and <em>R. berlandieri</em> late summer and into fall. The secondary peaks in breeding activity of allopatric <em>R. sphenocephala</em> and R. <em>berlandieri</em> have disappeared completely in sympatry (Figure <a href="speciation-1.html#fig:tempiso">11.4</a>). So, even as members of the three species occupy the same habitats, the likelihood of interbreeding is significantly reduced because they are not thinking about reproduction at the same time.</p>
<div class="figure"><span style="display:block;" id="fig:tempiso"></span>
<img src="Primer2Evolution_files/figure-html/tempiso-1.png" alt="Three species in the *Rana pipiensis* complex have largely overlapping breeding seasons when they occur in allopatry. However, when they coexist in the same location, breeding seasons shift such that each species is temporally isolated. [Data](data/11_temporal-iso.csv) from Hillis (1981)." width="100%" />
<p class="caption">
Figure 11.4: Three species in the <em>Rana pipiensis</em> complex have largely overlapping breeding seasons when they occur in allopatry. However, when they coexist in the same location, breeding seasons shift such that each species is temporally isolated. <a href="data/11_temporal-iso.csv">Data</a> from Hillis (1981).
</p>
</div>
<p>Finally, a form of ecological isolation that is important for reproductive isolation in plants is pollinator isolation. Pollinator isolation occurs when animal pollinators do not transfer pollen between plants of different populations, usually as a consequence of coevolution between floral traits and pollinator preferences. For example, two closely related species of columbines (<em>Aquilegia formosa</em> and <em>A. pubescens</em>) coexist in the same habitat, but they have different flower colors and morphologies that attract different species. <em>A. formosa</em> has red, hanging flowers, while <em>A. pubescens</em> has white, erect flowers (Figure <a href="speciation-1.html#fig:polliiso">11.5</a>A and B). Since the red flowers are primarily visited by hummingbirds and the white flowers by hawkmoths, pollen is primarily distributed from <em>A. formosa</em> to <em>A. formosa</em> and from <em>A. pubescens</em> to <em>A. pubenscens</em>, while interspecific fertilization is rare <a href="speciation-1.html#fig:polliiso">11.5</a>C). Other groups of plants have diverged in flower coloration along a yellow-red gradient, exploiting differences in the visual systems of insect pollinators (with biases for yellow and UV) and bird pollinators (with biases for red).</p>
<div class="figure"><span style="display:block;" id="fig:polliiso"></span>
<img src="Primer2Evolution_files/figure-html/polliiso-1.png" alt="A. *Aquilegia formosa*. Photo: Steve Berardi, [CC BY-SA 2.0](https://creativecommons.org/licenses/by-sa/2.0). B. *Aquilegia pubescens*. Photo: Dcrjsr, [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0). C. Due different pollinator preferences in flower color and morphology, cross-pollination between species of *Aquilegia* is relatively rare. [Data](data/11_pollinator-iso.csv) from Hodges and Arnold (1994)." width="100%" />
<p class="caption">
Figure 11.5: A. <em>Aquilegia formosa</em>. Photo: Steve Berardi, <a href="https://creativecommons.org/licenses/by-sa/2.0">CC BY-SA 2.0</a>. B. <em>Aquilegia pubescens</em>. Photo: Dcrjsr, <a href="https://creativecommons.org/licenses/by-sa/3.0">CC BY-SA 3.0</a>. C. Due different pollinator preferences in flower color and morphology, cross-pollination between species of <em>Aquilegia</em> is relatively rare. <a href="data/11_pollinator-iso.csv">Data</a> from Hodges and Arnold (1994).
</p>
</div>
</div>
<div id="behavioral-isolation" class="section level4 unnumbered hasAnchor">
<h4>Behavioral isolation<a href="speciation-1.html#behavioral-isolation" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Behavioral isolation occurs when individuals from different populations do not recognize each other as potential mates. For example, populations may differ in species-recognition mechanisms, where changes in a particular signal (<em>e.g.</em>, the chemical composition of a pheromone or the color for a visual signal) alters the perception of individuals as con- vs. heterospecific, or their attractiveness during mate choice. Behavioral isolation is evident in many species of birds (especially based on song), fishes (especially based on coloration), and insects (especially based on chemical signals).</p>
<p><em>Heliconius</em> butterflies are an example of behavioral isolation. Sympatric species are able to produce fertile offspring, but they typically do not hybridize because of assortative mating preferences. Males of the sympatric <em>H. cydno</em> (Figure <a href="speciation-1.html#fig:behiso">11.6</a>A) and <em>H. melpomene</em> (Figure <a href="speciation-1.html#fig:behiso">11.6</a>B), for example, almost exclusively court females of their own species (Figure <a href="speciation-1.html#fig:behiso">11.6</a>C). Hence, the existence of species-specific signals and matching mating preferences can significantly reduce the probability of gene flow among closely related species.</p>
<div class="figure"><span style="display:block;" id="fig:behiso"></span>
<img src="Primer2Evolution_files/figure-html/behiso-1.png" alt="A. *Heliconius cydno*. Photo: Greg Hume, [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0). B. *Heliconius melpomene*. Photo: Charles J. Sharp, [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0). C. Males of both species have assortative mating preferences and almost exclusively court females of their own species. [Data](data/11_heliconius.csv) from Jiggins et al.(2001)." width="100%" />
<p class="caption">
Figure 11.6: A. <em>Heliconius cydno</em>. Photo: Greg Hume, <a href="https://creativecommons.org/licenses/by-sa/3.0">CC BY-SA 3.0</a>. B. <em>Heliconius melpomene</em>. Photo: Charles J. Sharp, <a href="https://creativecommons.org/licenses/by-sa/4.0">CC BY-SA 4.0</a>. C. Males of both species have assortative mating preferences and almost exclusively court females of their own species. <a href="data/11_heliconius.csv">Data</a> from Jiggins et al.(2001).
</p>
</div>
</div>
<div id="mechanical-isolation" class="section level4 unnumbered hasAnchor">
<h4>Mechanical isolation<a href="speciation-1.html#mechanical-isolation" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Mechanical isolation occurs when individuals of two populations meet, recognize each other as potential mates, and attempt to copulate, but copulations are unsuccessful because of some mechanical incompatibility. Mechanical incompatibilities can be caused by size-mismatches between populations, or because of genital incompatibilities that arise through genital coevolution. Coevolution of male and female genital traits can arise through a variety of mechanisms, including sexual conflict (see <a href="social-behavior-and-sexual-selection.html#sexual-conflict">Chapter 10</a>). As consequence, genitals in many animal groups evolve and diversify rapidly, with stark differences even among closely related species. Compatibility of male and female genitalia have been likened to a lock-and-key mechanisms, where the males’ “key” will only fit species-specific female “locks”. Accordingly, heterospecific matings would not lead to successful sperm transfer and fertilization even when copulation is attempted. Mechanical isolation can also occur in plants, when closely related species have stamens and styles in different configurations. For example, black sage (<em>Salvia mellifera</em>) have stamens and style positioned along the upper lip of the flower; hence, pollen is transferred from flower to flower on the backs of small pollinating bees. In contrast, the syntopic white sage (<em>S. apiana</em>) exhibits two long stamens and a long, forked style that extends out and away from the flower. This configuration primarily transfers pollen to the wings of larger carpenter and bumble bees. So even if the same pollinator visits flowers of both species, successful transfer of pollen from one species to the style of the other species is unlikely.</p>
<p>Another fascinating example of mechanical isolation comes from evolution experiments using feather lice. These avian parasites must be small enough to fit between the feather barbs of their host to avoid being removed during preening. On the other hand, there is selection for larger body size due to the higher fecundity of larger individuals. Since larger hosts also have a larger distance between the barbs of their feathers, there is a correlation between host and parasite body size. Scott Villa and his colleagues investigated the evolution of feather lice (<em>Columbicola columbae</em>) on domesticated pigeons (<em>Columba livia</em>) of different body size over a period of four years (60 louse generations). As expected, lice that lived on large hosts evolved a larger body size over this period of time. Interestingly, this also led to the evolution of mechanical isolation between lice from large hosts and lice from small hosts. While males were still eager to mate with females of the other strain, size mismatched individuals were unable to copulate effectively with each other (check out the videos of a mating of a <a href="https://movie-usa.glencoesoftware.com/video/10.1073/pnas.1901247116/video-1">size-matched pair</a>, <a href="https://movie-usa.glencoesoftware.com/video/10.1073/pnas.1901247116/video-2">a male of the large strain trying to copulate with a female of the small strain</a>, and <a href="https://movie-usa.glencoesoftware.com/video/10.1073/pnas.1901247116/video-3">a male of the small strain trying to copulate with a female of the large strain</a>). These mismatches resulted in reduced copulation times, a reduced probability of egg production, and a reduced number of offspring produced in interlineage matings (Figure <a href="speciation-1.html#fig:copiso">11.7</a>). Hence, mechanical incompatibility has the capacity to reduce gene flow between populations adapted to different host sizes.</p>
<div class="figure"><span style="display:block;" id="fig:copiso"></span>
<img src="Primer2Evolution_files/figure-html/copiso-1.png" alt="Even though male feather lice (*Columbicola columbae*) of different sized lineages attempt to copulate with females that are too large or too small (as indicated by point outside of the gray shaded area), those copulations are not necessrily successful. A. Lice within the typical range of sexual size dimorphism exhibited longer copulation duration compared to matings between individuals with large size disparities. [Data](data/11_lice-copulation.csv) from Villa et al. (2019). B. Lice within the typical range of sexual size dimorphism were more likely to produced eggs when kept in experimental vials. [Data](data/11_lice-eggs.csv) from Villa et al. (2019). C. Lice within the typical range of sexual size dimorphism were more likely to produced offspring on a natural host. [Data](data/11_lice-reprod.csv) from Villa et al. (2019)." width="100%" />
<p class="caption">
Figure 11.7: Even though male feather lice (<em>Columbicola columbae</em>) of different sized lineages attempt to copulate with females that are too large or too small (as indicated by point outside of the gray shaded area), those copulations are not necessrily successful. A. Lice within the typical range of sexual size dimorphism exhibited longer copulation duration compared to matings between individuals with large size disparities. <a href="data/11_lice-copulation.csv">Data</a> from Villa et al. (2019). B. Lice within the typical range of sexual size dimorphism were more likely to produced eggs when kept in experimental vials. <a href="data/11_lice-eggs.csv">Data</a> from Villa et al. (2019). C. Lice within the typical range of sexual size dimorphism were more likely to produced offspring on a natural host. <a href="data/11_lice-reprod.csv">Data</a> from Villa et al. (2019).
</p>
</div>
</div>
</div>
<div id="post-mating-pre-zygotic-isolation" class="section level3 hasAnchor" number="11.3.2">
<h3><span class="header-section-number">11.3.2</span> Post-Mating, Pre-zygotic Isolation<a href="speciation-1.html#post-mating-pre-zygotic-isolation" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Post-mating, prezygotic isolation describes reproductive barriers that reduce the frequency or success of inter-population fertilization, if mating between members of different populations has occurred. Post-mating prezygotic isolation mechanisms can be associated with sperm competition and cryptic female choice, or—most importantly—with a gametic incompatibilities.</p>
<div id="gametic-isolation" class="section level4 unnumbered hasAnchor">
<h4>Gametic Isolation<a href="speciation-1.html#gametic-isolation" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Gametic isolation occurs when male and female gametes cannot merge to form a zygote. Such gametic incompatibilities are frequently mediated by interactions of proteins on the surface of eggs and sperm. Co-evolution of sperm and egg surface proteins can create a lock-and-key mechanism, just like co-evolution of male and female genitalia. Gametic isolation is particularly well-studied in aquatic broadcast spawners (<em>e.g.</em>, in Porifera, Cnidaria, and Echinodermata), where individuals release their gametes directly into the water, which of course limits the possibility for pre-mating isolation. For example, there are strong gametic incompatibilities among populations of a tropical sea urchin, <em>Echinometra mathaei</em> (Palumbi &amp; Metz 1991). Mixing eggs and sperm from different populations clearly indicates high fertilization rates when eggs are encountering sperm from the same population, irrespective of sperm concentration (Figure <a href="speciation-1.html#fig:urch">11.8</a>). In contrast, only 2-9 % of eggs were fertilized when sperm and eggs of different populations were combined. Hence, chemical interactions between eggs and sperm can be critical for determining the degree of reproductive isolation between populations. It is important to note that gametic isolation can also contribute to reproductive isolation between populations of species with other reproductive modes—including some with internal fertilization; however, mechanisms mediating gametic incompatibilities in those taxa are not as thoroughly investigated.</p>
<div class="figure"><span style="display:block;" id="fig:urch"></span>
<img src="Primer2Evolution_files/figure-html/urch-1.png" alt="xxx. [Data](data/11_sea-urchin.csv) from Palumbi and Metz (1991)." width="100%" />
<p class="caption">
Figure 11.8: xxx. <a href="data/11_sea-urchin.csv">Data</a> from Palumbi and Metz (1991).
</p>
</div>
</div>
</div>
<div id="post-zygotic-isolation" class="section level3 hasAnchor" number="11.3.3">
<h3><span class="header-section-number">11.3.3</span> Post-Zygotic Isolation<a href="speciation-1.html#post-zygotic-isolation" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Post-zygotic isolation occurs when individuals from different populations mate successfully and their gametes form a zygote that starts to develop, but the resulting hybrids have reduced fitness compared to either of the parents. Reduced hybrid fitness can be caused by both intrinsic and extrinsic factors.</p>
<div id="intrinsic-incompatibilities" class="section level4 unnumbered hasAnchor">
<h4>Intrinsic Incompatibilities<a href="speciation-1.html#intrinsic-incompatibilities" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Intrinsic hybrid incompatibilities are reductions in hybrid fitness that are independent of environmental conditions. Intrinsic barriers primarily arise due to epistatic effects and are also known as genetic incompatibilities. Such incompatibilities can be severe and cause hybrid inviability, where the development of hybrids ceases at early stages, long before such individuals are born. In other cases, genetic incompatibilities cause hybrid sterility (<em>e.g.</em>, sterility in mules, which are hybrids between horses and donkeys), or quantitative reductions in hybrid survival or reproduction.</p>
<p>Genetic incompatibilities can arise through coevolution of different genes that contribute to the same protein complex or the sample physiological pathway. For example, enzymes involved the mitochondrial respiratory chain are protein complexes composed of amino acid chains derived from both the nuclear and mitochondrial genomes, and different protein subunits need to be compatible and work together to form a functional protein. Evolutionary change in one subunit may consequently be accompanied by corresponding changes in other subunits. Breaking up such co-evolved gene complexes during hybridization, where an individual might inherit the mitochondria from one parent but nuclear subunits from another, can cause issues with protein complex assembly or function, ultimately leading to mitochondrial, cellular, and organismal dysfunctions. These cases of genetic incompatibilities are also known as mito-nuclear incompatibilities.</p>
<p>Another well-studied example of a genetic incompatibility occurs through the coevolution of tumor and tumor-suppression genes in swordtail fishes of the genus <em>Xiphophorus</em>. <em>X. malinche</em> has a tumor gene with low activity and a matching tumor-suppressor gene with low activity, while <em>X. birchmanni</em> has a tumor gene with high activity and a matching tumor suppressor gene with high activity (Powell et al. 2020). Hybridization can disrupt the balance between tumor genes and their suppressors. Particularly, <em>X. malinche</em> and <em>X. birchmanni</em> hybrids that inherit at least one high activity tumor allele from <em>X. birchmanni</em> (on chromosome 21) and only low activity tumor-suppressor genes from <em>X. malinche</em> (on chromosome 5) have an increased chance of developing invasive melanocytes, which cause melanoma (a form of skin cancer; Figure <a href="speciation-1.html#fig:xiph">11.9</a>A). Mismatches of tumor and tumor-suppressor genes in hybrids have clear consequences for hybrid phenotypes and fitness. For example, hybrids exhibit larger melanistic spots on their tails (Figure <a href="speciation-1.html#fig:xiph">11.9</a>B), and—consistent with the higher invasiveness of melanocytes—their caudal spots grow faster and larger over time (Figure <a href="speciation-1.html#fig:xiph">11.9</a>C). In addition, the frequency of fish with a caudal spot deceases across age classes in hybrid populations, indicating that individuals with spots are selected against (Figure <a href="speciation-1.html#fig:xiph">11.9</a>D). This decrease is particularly evident in hybrid populations with a higher caudal spot frequency (populations hybrid 2 and 3 in Figure <a href="speciation-1.html#fig:xiph">11.9</a>D), where the high-activity tumor gene derived from <em>X. birchmanni</em> is more common.</p>
<div class="figure"><span style="display:block;" id="fig:xiph"></span>
<img src="Primer2Evolution_files/figure-html/xiph-1.png" alt="A. Frequency of invasive melanocytes in *Xiphophorus* fish with different genotypes at a tumor suppressor locus (on chromosome 5) and tumor locus (on chromosome 21). Only fish that are homozygous for the low activity tumor suppressor allele from *X. malinche* and have at least one tumor allele from *X. birchmanni* develop a high frequency of invasive tumor cells. [Data](data/11_xipho1.csv) from Powell et al. (2020). B. Caudal spot frequencies in parental *X. birchmanni* and *X. malinche* populations, and natural populations with hybridization between the two species (Hybrid 1-3). [Data](data/11_xipho2.csv) from Powell et al. (2020). C. Caudal spots grow faster and larger in hybrids compared to *X. birchmanni*. [Data](data/11_xipho3.csv) from Powell et al. (2020). D. The decrease in caudal spot frequency from juveiles to adults in two populations of *Xiphophorus* hybrids indicate an increased mortality of fish with a caudal spot. [Data](data/11_xipho4.csv) from Powell et al. (2020)." width="100%" />
<p class="caption">
Figure 11.9: A. Frequency of invasive melanocytes in <em>Xiphophorus</em> fish with different genotypes at a tumor suppressor locus (on chromosome 5) and tumor locus (on chromosome 21). Only fish that are homozygous for the low activity tumor suppressor allele from <em>X. malinche</em> and have at least one tumor allele from <em>X. birchmanni</em> develop a high frequency of invasive tumor cells. <a href="data/11_xipho1.csv">Data</a> from Powell et al. (2020). B. Caudal spot frequencies in parental <em>X. birchmanni</em> and <em>X. malinche</em> populations, and natural populations with hybridization between the two species (Hybrid 1-3). <a href="data/11_xipho2.csv">Data</a> from Powell et al. (2020). C. Caudal spots grow faster and larger in hybrids compared to <em>X. birchmanni</em>. <a href="data/11_xipho3.csv">Data</a> from Powell et al. (2020). D. The decrease in caudal spot frequency from juveiles to adults in two populations of <em>Xiphophorus</em> hybrids indicate an increased mortality of fish with a caudal spot. <a href="data/11_xipho4.csv">Data</a> from Powell et al. (2020).
</p>
</div>
</div>
<div id="extrinsic-incompatibilities" class="section level4 unnumbered hasAnchor">
<h4>Extrinsic Incompatibilities<a href="speciation-1.html#extrinsic-incompatibilities" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Extrinsic incompatibilities occur when hybrids face reduced fitness because of a mismatch between their phenotype and their abiotic or biotic environment. In this case, hybrids are generally viable and fertile under laboratory conditions, but are ecologically inviable under natural conditions— either because they cannot tolerate the stressful environmental conditions that periodically arise in nature, or because they are competitively inferior to either parent in their respective niche. For example, some North American postglacial lakes harbor two species of stickleback (<em>Gasterosteus aculeatus</em>): a smaller limnetic species that lives in the open water and primarily feeds on zooplankton, and a larger benthic species that lives in the littoral zone and feeds on macroinvertebrates (Schluter 1995). Hybrids between the two species are viable and fertile. However, cage experiments in natural habitats revealed that hybrids are ecologically inferior compared to either parental species in the parents’ native habitat; hybrids have a lower growth rate than benthic stickleback in the littoral zone and also have a lower growth rate than the limnetic parents in open water (Figure <a href="speciation-1.html#fig:hybsel">11.10</a>A). Hence, ecological selection limits hybrid success in the natural context.</p>
<p>Hybrids may also be behaviorally sterile. Behavioral sterility may arise by hybrids not recognizing members of either parental species as potential mates, or because they themselves are not recognized as potential mates. In other words, behavioral sterility in hybrids is a consequence mismatched signals involved in species recognition, mate choice, and associated mating preferences. An example of reduced hybrid fitness caused by mate choice comes from the same <em>Heliconius</em> butterflies that we already discussed in the context of behavioral isolation. <em>Heliconius melpomene</em> females—and to a lesser degree those of <em>H. cydno</em>—exhibit mating preferences for males of their own species, as opposed to F1 hybrid males stemming from a cross between the two species (Figure <a href="speciation-1.html#fig:hybsel">11.10</a>B). Notably, however, females of both species also prefer to mate with hybrids over males from the opposite species, indicating that behavioral sterility is not complete.</p>
<div class="figure"><span style="display:block;" id="fig:hybsel"></span>
<img src="Primer2Evolution_files/figure-html/hybsel-1.png" alt="A. Ecological selection against hybrids as evidenced in stickleback (*Gasterosteus aculeatus*). Hybrids between benthic and limnetic stickleback have lower growth rates than either parent in their native habitat. [Data](data/11_hyb-sel.csv) from Schluter (1995). B. Sexual selection against hybrids as evidenced in *Heliconius* butterflies. Females of *H. cydno* and especially *H. melpomene* tend to avoid hybrid males during mate choice. [Data](data/11_heliconius-hybrid.csv) from Naisbit et al. (2001)." width="100%" />
<p class="caption">
Figure 11.10: A. Ecological selection against hybrids as evidenced in stickleback (<em>Gasterosteus aculeatus</em>). Hybrids between benthic and limnetic stickleback have lower growth rates than either parent in their native habitat. <a href="data/11_hyb-sel.csv">Data</a> from Schluter (1995). B. Sexual selection against hybrids as evidenced in <em>Heliconius</em> butterflies. Females of <em>H. cydno</em> and especially <em>H. melpomene</em> tend to avoid hybrid males during mate choice. <a href="data/11_heliconius-hybrid.csv">Data</a> from Naisbit et al. (2001).
</p>
</div>
</div>
</div>
<div id="reproductive-isolation-as-a-complex-trait" class="section level3 hasAnchor" number="11.3.4">
<h3><span class="header-section-number">11.3.4</span> Reproductive Isolation as a Complex Trait<a href="speciation-1.html#reproductive-isolation-as-a-complex-trait" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>It is important to note that different mechanisms of reproductive isolation are not mutually exclusive. Between any population pair, many mechanisms may coincide such that total reproductive isolation is a composite of mechanisms that reduce the probability of mating between different populations, reduce the probability of fertilization should mating happen, and reduce the success of hybrids should fertilization happen. More over, as we will see below, the evolution of one reproductive isolation mechanism can profoundly impact the evolution of other reproductive barriers. Hence, it many natural systems it is critical not to just examine different mechanisms in isolation, but to also consider how different mechanisms interact to keep gene pools separated from one another.</p>
<div class="figure"><span style="display:block;" id="fig:incomps"></span>
<img src="images/incompats.png" alt="Different mechanisms of reproductive isolation often act in concert and can influence each other evolutionarily." width="100%" />
<p class="caption">
Figure 11.11: Different mechanisms of reproductive isolation often act in concert and can influence each other evolutionarily.
</p>
</div>
</div>
</div>
<div id="the-evolution-of-reproductive-isolation" class="section level2 hasAnchor" number="11.4">
<h2><span class="header-section-number">11.4</span> The Evolution of Reproductive Isolation<a href="speciation-1.html#the-evolution-of-reproductive-isolation" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Now that we have examined what mechanisms can cause reproductive isolation between populations, we can ask how those isolating barriers actually arise. Understanding how different evolutionary forces impact the evolution of reproductive isolation ultimately sheds light on the speciation process. We will consider three distinct scenarios: (1) Polyploidization and the evolution of instantaneous reproductive isolation, (2) allopatric speciation that includes periods of geographical separation between populations, and (3) ecological speciation, where reproductive isolation between populations can arise as a byproduct of adaptation even in the presence of ongoing gene flow.</p>
<div id="polyploidization-and-instantaneous-speciation" class="section level3 hasAnchor" number="11.4.1">
<h3><span class="header-section-number">11.4.1</span> Polyploidization and Instantaneous Speciation<a href="speciation-1.html#polyploidization-and-instantaneous-speciation" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Polyploidization is a form of mutation that leads to the duplication of entire genomes, typically due to meiotic errors leading to unreduced gametes. Fertilization of unreduced gametes with other unreduced gametes causes the formation of fertile polyploid lineages (<em>i.e.</em>, from diploid to tetraploid, from tetraploid to octoploid, etc.) (Figure <a href="speciation-1.html#fig:polploid">11.12</a>A). Polyploid lineages are immediately isolated from their ancestral lineage because of dysfunctional chromosome complements in crosses between individuals of different ploidy. Such auto-polyploid speciation is particularly common in plants and other organisms for which self-fertilization is possible, because the combination of two unreduced gametes is a strong limiting factor in the formation of polyploids. Accordingly, up to 4 % of plant species are thought to have arisen through auto-polyploid speciation (Figure <a href="speciation-1.html#fig:polploid">11.12</a>B), while this phenomenon is rare in animals.</p>
<p>Polyploid speciation can also occur through the combination of two genomes derived from different species, a process called allo-polyploid speciation (Figure <a href="speciation-1.html#fig:polploid">11.12</a>C). In this case, an unreduced gamete of one species is fertilized with a normal gamete from another, creating a hybrid with an uneven number of chromosomes. When unreduced gametes from that hybrid are then fertilized by a normal gamete from the second species, a fertile hybrid with the full genome set of each parent arises. Allo-polyploidy is extremely common in plants, and about 80 % of plant species may be allo-polyploids by some estimations. The common bread wheat (<em>Triticum aestivum</em>), for example, is an allo-hexaploid that contains three distinct sets of chromosomes from three different species of grass in the genus <em>Aegilops</em>. Similarly, various vegetables in the cabbage family (mustard, collard greens, cauliflower, kale, bok choi, rape seed, etc.) represent different allo-polyploid combinations of three species, <em>Brassica nigra</em>, <em>B. rapa</em>, and <em>B. olarecea</em>. In contrast, allo-polyploids are rare among animals, but some cases have been documented in insects, fish, amphibians, and reptiles.</p>
<div class="figure"><span style="display:block;" id="fig:polploid"></span>
<img src="images/polyploidization.png" alt="A. Auto-polyploidization occurs through the combination of two unreduced gametes, which usually occirs through self-fertilization. The resulting ployploids are instantaneously isolated from ancestral lineages with lower ploidy. B. Blackberries of the genus *Rubus* represent a polyploid species complex, including species with 2, 4, 8, 16, and 32 sets of chromosomes. Without genetic or cytological analyses, species with different ploidy levels are almost impossible to distinguish. C. Allo-polyploidization oocurs through the combination of two genomes derived from different species. It is a multi-step process that first involves the production of hybrids with uneven chromosome sets." width="100%" />
<p class="caption">
Figure 11.12: A. Auto-polyploidization occurs through the combination of two unreduced gametes, which usually occirs through self-fertilization. The resulting ployploids are instantaneously isolated from ancestral lineages with lower ploidy. B. Blackberries of the genus <em>Rubus</em> represent a polyploid species complex, including species with 2, 4, 8, 16, and 32 sets of chromosomes. Without genetic or cytological analyses, species with different ploidy levels are almost impossible to distinguish. C. Allo-polyploidization oocurs through the combination of two genomes derived from different species. It is a multi-step process that first involves the production of hybrids with uneven chromosome sets.
</p>
</div>
</div>
<div id="allopatric-speciation-and-hybridization" class="section level3 hasAnchor" number="11.4.2">
<h3><span class="header-section-number">11.4.2</span> Allopatric Speciation and Hybridization<a href="speciation-1.html#allopatric-speciation-and-hybridization" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>The allopatric speciation model was first introduced by Ernst Mayr in his seminal book <em>Systematics and the Origin of Species</em>. This model postulates that the speciation process unfolds in three distinct steps: (1) The subdivision of an initial population through a geographic barrier, (2) population divergence in isolation, and (3) secondary contract of incipient species. In the following sections, we will examine these three steps in detail, along with pertinent examples.</p>
<div class="alert alert-block alert-info">
<p><b>Explore More: Systematics and the Origin of Species</b></p>
<p>Ernst Mayr’s book “<em>Systematics and the Origin of Species</em>” is available for free through <a href="https://archive.org/details/in.ernet.dli.2015.20284">archive.org</a>.</p>
</div>
<div id="geographic-isolation" class="section level4 unnumbered hasAnchor">
<h4>Geographic Isolation<a href="speciation-1.html#geographic-isolation" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>An initial stage of geographic isolation is central to the allopatric speciation model. Geographic isolation drives reproductive isolation and limits the spread of alleles within the separate populations. Two mechanisms can create geographically isolated populations from an initial ancestor: dispersal and vicariance.</p>
<p>Dispersal occurs when individuals of an original population overcome an existing geographic barrier and colonize a new and previously unoccupied habitat (Figure <a href="speciation-1.html#fig:dispersal">11.13</a>A). Classic examples of dispersal occur during the colonization of new oceanic islands. For example, the Hawai’ian archipelago arose as Earth’s crust moved over a volcanic hot spot from east to west. Accordingly, the western most island (Kaua’i; ~4.2–5 million years old) is the oldest, and islands get progressively younger towards the east; Hawai’i, as the youngest island, rose out of the ocean only 0.3–1.0 million years ago. <em>Drosophila</em> fruitflies that colonized the archipelago early during its formation provide a formidable example of dispersal. For two independent clades within the genus, the sequence of phylogenetic divergence corresponds to the sequence in which different islands arose from the ocean. In the <em>D. planitibia</em> subgroup (green clade in Figure <a href="speciation-1.html#fig:dispersal">11.13</a>B), dispersal occurred from O’ahu to Maui to Hawai’i; in the <em>D.</em> <em>mitchelli</em>, subgroup dispersal occurred from O’ahu to Moloka’i to Hawai’i. This pattern is consistent with the idea that species diversified through step-wise colonization of new habitats as they became available. Similar processes also gave rise to the diversity we observe in other oceanic archipelagos, like the Galapagos. It is important to emphasize that dispersal across geographic barriers and colonization of novel habitats is important in other ecological contexts as well. For example, lakes are the terrestrial equivalent of oceanic islands. Species may also overcome mountain ranges that separate suitable habitats, or valleys that separate mountain tops. Furthermore, island-like situations occur in the apparently well-connected habitats within oceans. For example, patches of coral reefs or deep-sea hydrothermal vents are separated by vast stretches of unsuitable habitats, which migrants must overcome.</p>
<div class="figure"><span style="display:block;" id="fig:dispersal"></span>
<img src="images/dispersal.png" alt="A. Dispersal is the colonization of a new habitat across a geographic barrier. B. Patterns of diversification in *Drosophila* fruitflies are consistent with dispersal. In two clades (green and orange), the sequence of species divergence corresponds to colonization of younger islands in the archipelago (adopted from Obbard et al. 2012). The inset photo shows *D. silvestris* (KarlM, [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0))." width="100%" />
<p class="caption">
Figure 11.13: A. Dispersal is the colonization of a new habitat across a geographic barrier. B. Patterns of diversification in <em>Drosophila</em> fruitflies are consistent with dispersal. In two clades (green and orange), the sequence of species divergence corresponds to colonization of younger islands in the archipelago (adopted from Obbard et al. 2012). The inset photo shows <em>D. silvestris</em> (KarlM, <a href="https://creativecommons.org/licenses/by-sa/3.0">CC BY-SA 3.0</a>).
</p>
</div>
<p>Vicariance occurs when the distributional range of a species is subdivided by an emerging geographic barrier (Figure <a href="speciation-1.html#fig:vicariance">11.14</a>A). For example, islands—or even entire continents—may be divided as a consequence of rising sea levels associated with climate change; as water levels flood low laying valleys, regions with higher elevations become separated. Similarly, volcanic activity and plate tectonics can shift land masses and create novel barriers of unsuitable habitat across which movement of organisms becomes limited. For example, the ancestor of ratites, which includes a number of flightless birds, had a Gondwanian distribution. As the supercontinent broke up into smaller pieces and landmasses drifted apart, geographic isolation gave rise to today’s flightless birds in South America, Africa, Australia and New Guinea, and New Zealand (Figure <a href="speciation-1.html#fig:vicariance">11.14</a>B). Other classic examples of vicariance include paired species of crustaceans, fish, and other marine animals that occur in either side of the Central American land bridge (i.e., sister species in the Pacific and the Caribbean) and in the Atlantic and the Mediterranean Sea.</p>
<div class="figure"><span style="display:block;" id="fig:vicariance"></span>
<img src="images/vicariance.png" alt="A. Vicariance is the subsetting of an original population by an emerging geographic barrier B. Patterns of diversification in ratites are a consequence of vicariance. The ancestral, contiguous range of this group on Gondwana was broken apart as a consequence of continental drift." width="100%" />
<p class="caption">
Figure 11.14: A. Vicariance is the subsetting of an original population by an emerging geographic barrier B. Patterns of diversification in ratites are a consequence of vicariance. The ancestral, contiguous range of this group on Gondwana was broken apart as a consequence of continental drift.
</p>
</div>
</div>
<div id="population-divergence" class="section level4 unnumbered hasAnchor">
<h4>Population Divergence<a href="speciation-1.html#population-divergence" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>The second step of the allopatric speciation model is population divergence in isolation. Such population divergence was long thought to be driven primarily by mutation and genetic drift; different sets of mutations arise and drift to fixation in each of the isolated populations, ultimately causing them to diverge. However, it is now well-recognized that natural and sexual selection also play a critical role in driving the differentiation of isolated populations. Differences in abiotic and biotic environmental conditions can drive local adaptation, potentially causing substantial phenotypic and genetic differences between populations. Thus, divergent selection can accelerate population divergence, though it is not requisite for allopatric speciation to occur.</p>
</div>
<div id="secondary-contact-and-reproductive-isolation" class="section level4 unnumbered hasAnchor">
<h4>Secondary Contact and Reproductive Isolation<a href="speciation-1.html#secondary-contact-and-reproductive-isolation" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>The fate of populations that diverged in isolation is determined in the third stage of the allopatric speciation model. When populations come into secondary contact after the initial period of isolation, there is an entire range of potential outcomes. Divergent populations are often referred to as incipient species at this stage.</p>
<p>If divergence between incipient species was relatively minor and does not affect their ability to interbreed, secondary contact may simply cease the speciation process. In this case, the two lineages hybridize and fuse back together into a single panmictic population.</p>
<p>Alternatively, at least some degree of reproductive isolation may have accrued between incipient lineages. Consistent with the idea that drift was critical in driving population divergence, genetic incompatibilities were long thought to be a primary mechanism of reproductive isolation. In particular, secondary contact and hybridization between incipient species brings together alleles that previously drifted to fixation in the isolated lineages, and some of the new allelic combinations can be deleterious due to negative epistatic interactions (Figure <a href="speciation-1.html#fig:dmincomp">11.15</a>). Such genetic incompatibilities that reduce hybrid fitness upon secondary contact are also known as Dobzhansky-Muller incompatibilities.</p>
<p>There are two more important points to make here: (1) Dobzhansky-Muller incompatibilities do not only arise as a consequence of population differences that evolved through genetic drift. Alternate alleles that rise to high frequencies in different populations through divergent selection can also be subject to negative epistatic interactions and cause reductions in hybrid fitness. So, while the Dobzhansky-Muller model was primarily developed under the assumption that mutation and genetic drift are the primary drivers of population divergence, the model also applies when other evolutionary forces contribute to population differentiation. (2) Reproductive isolation between incipient species can not only be mediated by genetic incompatibilities (intrinsic post-zygotic isolation), but by any other mechanism of reproductive isolation as well. Evolution in isolation may have inadvertent consequences on ecological, behavioral, and morphological traits, which then affect the probability of mating and successful fertilization between incipient species. In addition, though there may be no genetic incompatibilities that limit hybrid success, hybrids may still be selected against by natural or sexual selection.</p>
<div class="figure"><span style="display:block;" id="fig:dmincomp"></span>
<img src="images/dm-incomps.png" alt="Dobzhansky-Muller incompatibilities arise as a consequence of evolution in geographic isolation. Specifically, new sets of mutations arise in geographic isolation and eventually drift to fixation. Upon secondary contact, recombination of these new alleles in hybrids can be subject to negative epistatic interactions. The novel alleles a and b have no evolutionary history of interacting, and if the alleles are incompatible, hybrid fitness is reduced." width="100%" />
<p class="caption">
Figure 11.15: Dobzhansky-Muller incompatibilities arise as a consequence of evolution in geographic isolation. Specifically, new sets of mutations arise in geographic isolation and eventually drift to fixation. Upon secondary contact, recombination of these new alleles in hybrids can be subject to negative epistatic interactions. The novel alleles a and b have no evolutionary history of interacting, and if the alleles are incompatible, hybrid fitness is reduced.
</p>
</div>
<p>Irrespective of what mechanisms mediate reproductive isolation between incipient species, reproductive isolation is rarely complete upon secondary contact, as evidenced by the commonality of hybridization in nature. Hybridization, in these cases, is not evidence that incipient species are not species, but rather that the speciation process is still ongoing and has yet to be completed. Ultimately, there are three alternative outcomes upon secondary contact besides complete fusion of the the incipient species: completion of the speciation process through reinforcement, stable hybrid zones of contact, and hybrid speciation.</p>
<div id="reinforcement-and-the-completion-of-the-speciation-process" class="section level5 unnumbered hasAnchor">
<h5>Reinforcement and the Completion of the Speciation Process<a href="speciation-1.html#reinforcement-and-the-completion-of-the-speciation-process" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Reductions of hybrid fitness upon secondary contract—whether caused by genetic incompatibilities or by natural or sexual selection against hybrids—should exert direct selection on assortative mating preferences in both incipient species. Individuals that exhibit preferences for mating partners from their own incipient species and discriminate against heterospecifics ultimately have higher fitness, because they avoid producing unfit hybrid offspring. The process of natural selection increasing the degree of reproductive isolation between incipient species is known as reinforcement. In practice, reinforcement often leads to reproductive character displacement, where differences in reproductive traits—both in terms of signals and the corresponding preferences—are accentuated whenever closely related species coexist. A particular trait may be similar between two species in allopatry, but starkly differentiated in sympatry (Figure <a href="speciation-1.html#fig:reinforcement">11.16</a>). In the end, reproductive character displacement strengthens pre-zygotic reproductive isolation and can lead to the completion of the speciation process.</p>
<div class="figure"><span style="display:block;" id="fig:reinforcement"></span>
<img src="images/Character_Displacement.png" alt="Reinforcement can lead to reproductive character displacement, where differences between species are accentuated in sympatry." width="100%" />
<p class="caption">
Figure 11.16: Reinforcement can lead to reproductive character displacement, where differences between species are accentuated in sympatry.
</p>
</div>
<p>The importance of reinforcement in maintaining species boundaries upon secondary contact has been studied in <em>Ficedula</em> flycatchers, a group of small, Old World songbirds. Allopatric populations of the collared flycatcher (<em>F. albicollis</em>) and the pied flycatcher (<em>F. hypoleuca</em>) both exhibit black and white plumage and look almost identical to the untrained eye (Figure <a href="speciation-1.html#fig:piefly">11.17</a>A and B). However, the two species are very easy to tell apart when they co-occur in sympatry, because the pied flycatchers are actually brown and white (Figure <a href="speciation-1.html#fig:piefly">11.17</a>C). A series of elegant behavioral experiments has revealed how these changes in plumage coloration are a consequence of reinforcement (Sætre et al. 1997). It turns out that plumage similarities not only make it difficult for humans to distinguish allopatric populations of collared and pied flycatchers; one third of matings actually occur <em>between</em> species when females from sympatric populations have a choice between the allopatric color morphs of the two species. This finding is in stark contrast to the strong assortative mating observed when females can choose between males of sympatric color morphs (Figure <a href="speciation-1.html#fig:piefly">11.17</a>D). Consequently, differences in plumage coloration reduce the probability of hybridization, as predicted by the reinforcement hypothesis. Moreover, experiments with female pied flycatchers have revealed evidence for the evolution of mating preferences between sympatric and allopatric populations. While sympatric females prefer males with the brown plumage of sympatric males, allopatric females prefer to mate with males that exhibit the characteristically black plumage of allopatric males (Figure <a href="speciation-1.html#fig:piefly">11.17</a>E). Hence, reinforcement has not only acted on a signal for species recognition and caused character displacement in male plumage coloration, but it has also acted on the corresponding female preference to increase pre-zygotic reproductive isolation.</p>
<div class="figure"><span style="display:block;" id="fig:piefly"></span>
<img src="Primer2Evolution_files/figure-html/piefly-1.png" alt="A. Collared flycatcher (*Ficedula albicollis*). Photo: [Andrej Chudy](https://www.flickr.com/photos/andrej_chudy), [CC BY-SA 2.0](https://creativecommons.org/licenses/by-sa/2.0). B. Allopatric pied flycatchers (*Ficedula hypoleuca*) are black and white like collared flycatchers. Photo: Francesco Veronesi, [CC BY-SA 2.0](https://creativecommons.org/licenses/by-sa/2.0). C. Sympatric pied flycatchers (*Ficedula hypoleuca*) are brown and white. Photo: Steve Garvie, [CC BY-SA 2.0](https://creativecommons.org/licenses/by-sa/2.0). D. While there is strong assortative mating when female flycatchers have a choice between males of the two species that exhibit the sympatric color combination, nearly a third of matings are heterospecific when females are given a choice between males from the two species that exhibit the allopatric color combination. [Data](data/11_flycatcher_spec_rec.csv) from Sætre et al. (1997). E. Evidence for the evolution of female mating preferences in pied flycatchers. Females from sympatric populations prefer to mate with males that exhibit the brown plumage typical for sympatric males; females from allopatric populations prefer males with the black plumage typical for allopatric males. [Data](data/11_flycatcher_mate_pref.csv) from from Sætre et al. (1997)." width="100%" />
<p class="caption">
Figure 11.17: A. Collared flycatcher (<em>Ficedula albicollis</em>). Photo: <a href="https://www.flickr.com/photos/andrej_chudy">Andrej Chudy</a>, <a href="https://creativecommons.org/licenses/by-sa/2.0">CC BY-SA 2.0</a>. B. Allopatric pied flycatchers (<em>Ficedula hypoleuca</em>) are black and white like collared flycatchers. Photo: Francesco Veronesi, <a href="https://creativecommons.org/licenses/by-sa/2.0">CC BY-SA 2.0</a>. C. Sympatric pied flycatchers (<em>Ficedula hypoleuca</em>) are brown and white. Photo: Steve Garvie, <a href="https://creativecommons.org/licenses/by-sa/2.0">CC BY-SA 2.0</a>. D. While there is strong assortative mating when female flycatchers have a choice between males of the two species that exhibit the sympatric color combination, nearly a third of matings are heterospecific when females are given a choice between males from the two species that exhibit the allopatric color combination. <a href="data/11_flycatcher_spec_rec.csv">Data</a> from Sætre et al. (1997). E. Evidence for the evolution of female mating preferences in pied flycatchers. Females from sympatric populations prefer to mate with males that exhibit the brown plumage typical for sympatric males; females from allopatric populations prefer males with the black plumage typical for allopatric males. <a href="data/11_flycatcher_mate_pref.csv">Data</a> from from Sætre et al. (1997).
</p>
</div>
</div>
<div id="hybrid-zones" class="section level5 unnumbered hasAnchor">
<h5>Hybrid Zones<a href="speciation-1.html#hybrid-zones" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Secondary contact between incipient species can lead to stable hybrid zones in geographically restricted areas where lineages meet and cross-breed. Stable hybrid zones can either be maintained in so-called “tension zones”, or in areas of bounded hybrid superiority.</p>
<p>Tension zones are geographic regions where continuous dispersal and hybridization between parental species is balanced by selection against hybrids. Although hybrids are continuously produced, the integrity of species boundaries is largely maintained, because hybrids have lower fitness than either parental species and are continuously selected against. It is important to note, however, that backcrossing between hybrids and parentals can lead to the the introgression of neutral and adaptive alleles between species.</p>
<p>In contrast, the bounded hybrid superiority hypothesis postulates that hybrids have a higher fitness than either of the parental species in environments that are intermediate to those typically occupied by the parentals. In this case, hybrid zones are expected particularly along environmental gradients that separate the niches of the parental species. For example, hybrids between the two swordtail fishes <em>X. malinche</em> (a high elevation species) and <em>X. birchmanni</em> (a low elevation species) primarily occur at intermediate elevations. Hybrids have a higher heat tolerance than <em>X. malinche</em> and a higher cold tolerance than <em>X. birchmanni</em>, which allows them to outcompete either of the parental species at at intermediate elevations, where it gets too hot for <em>X. malinche</em> in the summer and too cold for <em>X. birchmanni</em> in the winter.</p>
</div>
<div id="hybrid-speciation" class="section level5 unnumbered hasAnchor">
<h5>Hybrid Speciation<a href="speciation-1.html#hybrid-speciation" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>Last but not least, hybridization upon secondary contact can lead to homoploid hybrid speciation; <em>i.e.</em>, the formation a new species, reproductively isolated from both parentals but without a change in ploidy levels. While not common, such hybrid speciation has been documented in insects, fishes, amphibians, birds, and even a marine mammal (the clymene dolphin, <em>Stenella clymene</em>, which arose through hybridization between spinner dolphin, <em>S. longirostris</em>, and striped dolphin, <em>S. coeruleoalba</em>).</p>
<p>Another notable example of hybrid speciation includes the Darwin’s finches on Daphne Major Island in the Galapagos Archipelago, which we studied in detail in <a href="evidence-for-evolution.html#case-study-darwins-finches">Chapter 2</a> and <a href="/a-mechanism-for-change.html#darwins-logic">Chapter 3</a>. In the 1980s, a male Española cactus finch (<em>Geospiza conirostris</em>) dispersed to Daphne Major and bred with female medium ground finches (<em>G. fortis</em>) that were already there. This hybridization event gave rise to a new lineage (also known as big bird lineage due to its size), which is endemic to Daphne Major and reproductively isolated from the other species that occur on the island.</p>
<div class="alert alert-block alert-info">
<p><b>Explore More: Big Bird and Hybrid Speciation</b></p>
<p>To learn more about the fantastic tale of hybrid speciation in Darwin’s finches as directly observed by humans, check out Jordana Cepelewicz’ article “<a href="https://www.quantamagazine.org/new-bird-species-arises-from-hybrids-as-scientists-watch-20171213/">New Bird Species Arises from Hybrids, as Scientists Watch</a>” in Quanta Magazine.</p>
</div>
</div>
</div>
</div>
<div id="ecological-speciation" class="section level3 hasAnchor" number="11.4.3">
<h3><span class="header-section-number">11.4.3</span> Ecological Speciation<a href="speciation-1.html#ecological-speciation" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>The allopatric speciation model dominated our understanding of the speciation process for over half a century. But with the rise of genetic tools, there was increasing evidence for sister species that occur in the same location without apparent periods of geographic separation. New species—it seemed—were also arising without physical separation, a process known as sympatric speciation. The big question, however, was how reproductive isolation can evolve between diverging populations when ongoing gene flow keeps homogenizing gene pools.</p>
<p>Both theoretical models and empirical evidence have since revealed that speciation with gene flow is possible if there is strong divergent selection driving sympatric populations apart. When selection is sufficiently strong, it can overcome the homogenizing effects of gene flow, leading to significant population differentiation (Figure <a href="speciation-1.html#fig:msdspec">11.18</a>). Hence, speciation becomes a mere consequence of adaptation to different environmental conditions. Speciation with gene flow in response to divergent selection is also known as ecological speciation.</p>
<p>It is important to note that ecological speciation does not only happen in sympatry, but also in allopatry and other geographical contexts (peri- and parapatry). Hence, the geographic context is less of a focus in ecological speciation research. Rather, research primarily focuses on how exactly adaptation in response to divergent selection translates to reproductive isolation between incipient species.</p>
<div class="figure"><span style="display:block;" id="fig:msdspec"></span>
<img src="Primer2Evolution_files/figure-html/msdspec-1.png" alt="Divergence between populations connected by gene flow is a reflection of the balance between divergent selection and the rate of migration. As selection gets stronger, the homogenizing effect of migration gets smaller. Each simulation conists of two populations (represented in red and blue) that are connected by migration at the rate of 0.05 per generation. However, similations differ in the strength of divergent selection. A. *w*&lt;sub&gt;migrants&lt;/sub&gt;=0.95, *w*&lt;sub&gt;hybrids&lt;/sub&gt;=0.975; B. *w*&lt;sub&gt;migrants&lt;/sub&gt;=0.8, *w*&lt;sub&gt;hybrids&lt;/sub&gt;=0.9; C. *w*&lt;sub&gt;migrants&lt;/sub&gt;=0.6, *w*&lt;sub&gt;hybrids&lt;/sub&gt;=0.8; D. *w*&lt;sub&gt;migrants&lt;/sub&gt;=0.25, *w*&lt;sub&gt;hybrids&lt;/sub&gt;=0.5." width="100%" />
<p class="caption">
Figure 11.18: Divergence between populations connected by gene flow is a reflection of the balance between divergent selection and the rate of migration. As selection gets stronger, the homogenizing effect of migration gets smaller. Each simulation conists of two populations (represented in red and blue) that are connected by migration at the rate of 0.05 per generation. However, similations differ in the strength of divergent selection. A. <em>w</em><sub>migrants</sub>=0.95, <em>w</em><sub>hybrids</sub>=0.975; B. <em>w</em><sub>migrants</sub>=0.8, <em>w</em><sub>hybrids</sub>=0.9; C. <em>w</em><sub>migrants</sub>=0.6, <em>w</em><sub>hybrids</sub>=0.8; D. <em>w</em><sub>migrants</sub>=0.25, <em>w</em><sub>hybrids</sub>=0.5.
</p>
</div>
<div id="reproductive-isolation-as-a-byproduct-of-adaptation" class="section level4 unnumbered hasAnchor">
<h4>Reproductive Isolation as a Byproduct of Adaptation<a href="speciation-1.html#reproductive-isolation-as-a-byproduct-of-adaptation" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>A wide variety of divergent sources of selection have been identified as driving forces in ecological speciation. For example, divergent selection can be caused by environmental differences in habitat structure, climate, or resource availability, as well as different communities of competitors, predators, or parasites. In addition, divergent selection may be driven by sexual selection, particularly when environmental variation causes selection on secondary sexual traits or on sensory systems.</p>
<p>Reproductive isolation during ecological speciation evolves incidentally, as a byproduct of adaption caused by divergent selection. Adaptation to different environmental conditions simply reduces the likelihood that individuals from diverging populations meet, that they recognize each other as potential mates if they do meet, that they are compatible during mating if they attempt to, and that they produce competitive hybrid offspring if eggs get fertilized. With the exception of gametic isolation, ecological speciation has been associated with every possible mechanism of reproductive isolation described above.</p>
<p>A classic example of ecological speciation stems from mosquitofish (<em>Gambusia hubbsi</em>) that inhabit blue holes on the Bahamas islands. Blue holes are water-filled sinkholes that were created when underground caverns collapsed. Some of the Bahamas islands are dotted with isolated blue holes (Figure <a href="speciation-1.html#fig:bahagamb">11.19</a>A), and most have been colonized by mosquitofish. These blue holes, however, differ in whether other fish species have also arrived. In some, the mosquitofish live by themselves; in others, they have to share their habitat with bigmouth sleepers (<em>Gobiomorus dormitor</em>), a large predatory fish. Mosquitofish in different blue holes have adapted to the local ecological conditions. In habitats without predators, mosquitofish have convergently evolved streamlined bodies that allow for energy efficient swimming, while those in habitats with predators have evolved bodies that allow for rapid acceleration during predator escape movements. How has adaptation to the different predation regimes impacted the emergence of reproductive isolation? Brian Langerhans and colleagues (2007) conducted mate choice experiments with mosquitofish from different ecological settings. They first quantified the strength of female preference for a male of their own population, as opposed to a male from a different blue hole with the same predator regime, or a male from a different blue hole with the opposite predator regime. The results indicated that assortative mating (i.e., a preference for the male from the female’s own population) was much stronger when females were confronted with a foreign male derived from different environmental conditions (Figure <a href="speciation-1.html#fig:bahagamb">11.19</a>B). So, females from no-predation habitats preferred to mate with males from no-predation habitats, while those from blue holes with bigmouth sleeper preferred to mate with males from other high-predation habitats. Females made mating decisions based on male traits that are directly related to adaptation to different predation environments, as evidenced by the negative correlation between morphological dissimilarity and the relative strength in mating preference (Figure <a href="speciation-1.html#fig:bahagamb">11.19</a>C). In other words, females from both environments prefer to mate with males exhibiting a body shape that matches their specific ecological context. Adaptive evolution of a morphological trait consequently had inadvertent repercussions for mate choice, and ultimately caused behavioral isolation between diverging lineages.</p>
<div class="figure"><span style="display:block;" id="fig:bahagamb"></span>
<img src="Primer2Evolution_files/figure-html/bahagamb-1.png" alt="A. Watling's Blue Hole, southwestern San Salvador Island, eastern Bahamas. Photo: [James St. John](https://www.flickr.com/people/47445767@N05), [CC BY 2.0](https://creativecommons.org/licenses/by/2.0). B. Assortative mating preferences of female mosquitofish that could choose beteen a male from their own population or a male that either came from a different blue hole with the same predation environment or a different blue hole from a different predation environment. Assortative mating with the male from the female's own population is much strong when she is confronted with a foreign male from a different environment. [Data](data/11_gambusia-assortative-mating.csv) form Langerhans et al. (2007). C. Female mating preferences are parimarily based on body shape, and relative mating preferences decrease with increasing dissimilarity of a male's body shape to the locally adapted form. [Data](data/11_gambusia-assortative-mating2.csv) form Langerhans et al. (2007)." width="100%" />
<p class="caption">
Figure 11.19: A. Watling’s Blue Hole, southwestern San Salvador Island, eastern Bahamas. Photo: <a href="https://www.flickr.com/people/47445767@N05">James St. John</a>, <a href="https://creativecommons.org/licenses/by/2.0">CC BY 2.0</a>. B. Assortative mating preferences of female mosquitofish that could choose beteen a male from their own population or a male that either came from a different blue hole with the same predation environment or a different blue hole from a different predation environment. Assortative mating with the male from the female’s own population is much strong when she is confronted with a foreign male from a different environment. <a href="data/11_gambusia-assortative-mating.csv">Data</a> form Langerhans et al. (2007). C. Female mating preferences are parimarily based on body shape, and relative mating preferences decrease with increasing dissimilarity of a male’s body shape to the locally adapted form. <a href="data/11_gambusia-assortative-mating2.csv">Data</a> form Langerhans et al. (2007).
</p>
</div>
</div>
<div id="the-role-of-sexual-selection" class="section level4 unnumbered hasAnchor">
<h4>The Role of Sexual Selection<a href="speciation-1.html#the-role-of-sexual-selection" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Even though ecological speciation is primarily driven by natural selection, sexual selection can also play a critical role in speciation processes. Specifically, ecological speciation can be facilitated when traits under divergent selection (<em>i.e</em>., traits involved in adaptation) also contribute to non-random mating (<em>i.e.</em>, behavioral isolation). Such traits are often referred to as “magic traits”, because a single trait is associated with functions usually attributed to separate traits—those mediating adaptation and those mediating assortative mating.</p>
<p>Magic traits appear to be a common theme in ecological speciation, as many adaptive modifications are also reliable cues for mating decisions. One of my favorite examples comes again from Darwin’s finches, where adaptive evolution of beak size has inadvertently affected the sounds males can produce when singing (Podos 2001). Adaptation to different food resources caused the diversification of beak sizes in finches from the genera <em>Camarhynchus</em> and <em>Geospiza</em>; species with smaller beaks tend to forage insects and other soft prey items, while those with larger beaks primarily feed on seeds. Changes in beak size have also impacted the vocal range individuals can produce. Birds with larger beaks have evolved songs with lower rates of syllable repetition and narrower frequency bandwidths, which ultimately reduces song complexity (Figure <a href="speciation-1.html#fig:beaksound">11.20</a>). Adaptation to different food resources was thus inadvertently correlated with the vocal communication essential in songbird mate recognition and courtship, facilitating the emergence of reproductive isolation in conjunction with adaptive trait evolution.</p>
<div class="figure"><span style="display:block;" id="fig:beaksound"></span>
<img src="Primer2Evolution_files/figure-html/beaksound-1.png" alt="Beak size evolution in Darwin's finches in correlated with song complexity, as measured by vocal deviation. Higher vocal deviation in this case corresponds to lower song complexity. [Data](data/11_beaksound.csv) from Podos (2001)." width="100%" />
<p class="caption">
Figure 11.20: Beak size evolution in Darwin’s finches in correlated with song complexity, as measured by vocal deviation. Higher vocal deviation in this case corresponds to lower song complexity. <a href="data/11_beaksound.csv">Data</a> from Podos (2001).
</p>
</div>
<p>Magic traits can also arise from selection on sensory systems that impact mating decisions, not just mating signals themselves. For example, cichlids in Lake Victoria have adapted their eye physiology to optimize visual perception at the different depths where they live (Seehausen et al. 2008). Since ambient light becomes more red-shifted at greater depths, cichlids have evolved retinal pigments that are more apt to detect red light. As a consequence of chooser biases (<a href="social-behavior-and-sexual-selection.html#chooser-biases">Chapter 10</a>), the changes in visual physiology are correlated with red shift in male nuptial coloration, such that adaptation to different light regimes has caused speciation between a population with blue nuptial coloration inhabiting shallow waters and a population with red nuptial coloration inhabiting deeper waters. Interestingly, speciation into blue and red forms has independently occurred multiple times within the lake.</p>
</div>
</div>
</div>
<div id="adaptive-radiation" class="section level2 hasAnchor" number="11.5">
<h2><span class="header-section-number">11.5</span> Adaptive Radiation<a href="speciation-1.html#adaptive-radiation" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Adaptation and speciation are intimately linked processes; hence, the same evolutionary forces that cause organismal adaptation also drive the evolution of biodiversity. In some instances, adaptation to novel niches and speciation occur rapidly and at large scales, causing a single ancestor to diversify into a large number of derived species with unique ecological adaptations. This process is known as adaptive radiation. According to Schluter (2000), four criteria must be fulfilled to confirm an adaptive radiation: (1) <em>Common ancestry</em>: All species in an adaptive radiation must be derived from a recent common ancestor. (2) <em>Phenotype-environment correlation</em>: There should be an association between the niches organisms occupy and the phenotypic traits used to exploit those niches. (3) <em>Trait utility</em>: Traits that are correlated with niches need to provide a fitness advantage to their carriers in their specific niche. (4) <em>Rapid speciation</em>: There needs to be a burst of speciation that coincides with the ecological and phenotypic diversification of a group.</p>
<p>Adaptive radiations are triggered when there is a wealth of ecological opportunity (open niches) that cannot be readily colonized by species already occupying those niches elsewhere. For example, adaptive radiations often occur in isolated lakes and on oceanic islands, where individuals of a single or a few colonizing species settle into niches that are occupied by a wealth of other species elsewhere. Similarly, adaptive radiations are evident in the fossil record after mass extinction events, where surviving species capitalized on open niches that were previously occupied. For example, mammals were able to diversity into novel niches that were freed up with the dinosaur extinction at the KT-boundary. Ecological opportunities may also arise through the evolution of novel traits (so-called key innovations) that allow organisms to enter novel adaptive zones that are yet unoccupied. The evolution of flight in birds and the evolution of endothermy in birds and mammals are key innovations that have allowed these taxa to invade new niches that were previously unavailable to ectotherms.</p>
<p>Evidence for adaptive radiations comes from a diversity of taxa. Among the best-known are Darwin’s finches, which adapted to different food resources on the Galapagos islands, honeycreepers (Fringillidae), fruitflies (<em>Drosophila</em>), and silverswords (<em>Argyroxiphium</em>) have radiated to exploit different niches on the Hawai’ian archipelago, and <em>Anolis</em> lizard exhibit parallel adaptive radiations on different Caribbean islands. The most stunning adaptive radiations, however, are found in African lakes, where cichlid fishes (family Cichlidae) have diversified into hundreds of species with unique adaptations. The largest adaptive radiations are found in Lake Victoria (&gt;500 species in as little as 14,000 years), Lake Tanganyika (about 200 species in 9-12 million years), and Lake Malawi (about 1,000 species in 3.4 million years). These are phenomenal numbers for single lakes, especially considering that there are more species of cichlids in African lakes than there are fish species in North America. How is it possible that so many species have evolved in such little time?</p>
<div id="african-cichlid-fishes" class="section level3 hasAnchor" number="11.5.1">
<h3><span class="header-section-number">11.5.1</span> African Cichlid Fishes<a href="speciation-1.html#african-cichlid-fishes" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>To learn more about the process of adaptive radiation, we will take a closer look at the adaptive radiations of cichlids in Lakes Victoria, Tanganyika, and Malawi. All of these lakes are huge; 3 of the top-10 largest lakes in the world. They are so big, in fact, that Livingston—on his expedition to cross the African continent—thought that he reached the Indian Ocean when he encountered Lake Malawi in 1859; imagine the disappointment when he first tasted the lake’s freshwater. Lake Victoria is very shallow, very young (400,000 years, although the lake was completely dry as recently as 14,000 years ago), and primarily located in Uganda and Tanzania. Lake Tanganyika and Lake Malawi are located further south and are associated with the African Rift, a tectonic spreading zone that divides the continent. Both lakes are much older than Lake Victoria and very deep (100 m for Lake Tanganyika, 770 m for Lake Malawi), although lake levels have fluctuated massively through time.</p>
<p>Considering the size of the lakes, it is not surprising that they offer a diversity of niches. When the lakes were colonized by cichlids, diversification primarily occurred along three ecological axes: (1) <em>Macrohabitat use</em>: The initial diversification in each lake likely occurred as species adapted to different macrohabitats. Some examples of macrohabitats include sandy beaches, rocky reefs, and open and deep water. (2) <em>Trophic resource use</em>: Within macrohabitats, species diverged in the food resources they exploit. Modifications to jaws and dentition have allowed different species to specialize on very specific diets: there are different types of piscivores (fish-eating predators), zooplankton feeders, molluscivores that can crush the shells of snails, crevice feeders with specialized jaws, algae eaters that either gather their food or scrape it off rocks, scale-eaters that pick scales off of other fish, fin biters, ectoparasite cleaners, and so-called paedophages that specialize on eating eggs and developing young that many cichlid females incubate in their mouths. (3) <em>Communication</em>: Species with similar habitat use and trophic specialization often diverge in communication, with stark divergence in male nuptial coloration and matching female preferences. The divergence along multiple niche axes allows even ecologically similar species to coexist. In general, there are four key factors that have contributed to the massive and repeated radiations of cichlid fishes.</p>
<div id="ecological-opportunity-and-key-innovation" class="section level4 unnumbered hasAnchor">
<h4>Ecological Opportunity and Key Innovation<a href="speciation-1.html#ecological-opportunity-and-key-innovation" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Comparative analyses beyond the three lakes we have discussed so far provided evidence for the role of ecological opportunity in adaptive radiation. Other African lakes also harbor adaptive radiations of cichlid fishes, although they are not quite as large—and often substantially smaller—than the radiations seen in Lakes Victoria, Tanganyika, and Malawi. Comparing lake size with the number of species in each adaptive radiation clearly indicates that larger lakes—which presumably have more ecological niches—also harbor a larger number of species (Figure <a href="speciation-1.html#fig:lakesize">11.21</a>). Hence, the sheer number of species in the Great African lakes is in part a consequence of their size.</p>
<p>Still, you might ask, why is it always cichlid fishes that have radiated in African lakes? A key innovation of cichlid fishes, which is hypothesized to have facilitated adaptation to different trophic niches, is the complexity of their jaws and the decoupling of oral and pharyngeal jaws. Cichlid do not have only one set of jaws visible from the outside (the oral jaws), but also a second internal set of jaws located at the back of the throat (pharyngeal jaws). Oral jaws are used for prey acquisition, while pharyngeal jaws are used for prey processing (e.g., shedding for plant matter or crushing of snails). The increased modularity of the jaw apparatus and the decoupling of different functions is thought to have lifted constraints in jaw evolution, making it possible for cichlids to modify their jaws in wide variety of ways and adapt to different food resources.</p>
<div class="figure"><span style="display:block;" id="fig:lakesize"></span>
<img src="Primer2Evolution_files/figure-html/lakesize-1.png" alt="The relationship of lake size and species number in different adaptive radiations of cichlid fishes. [Data](data/11_cichlid-lakesize.csv) from Seehausen (2006)." width="100%" />
<p class="caption">
Figure 11.21: The relationship of lake size and species number in different adaptive radiations of cichlid fishes. <a href="data/11_cichlid-lakesize.csv">Data</a> from Seehausen (2006).
</p>
</div>
</div>
<div id="hybridization" class="section level4 unnumbered hasAnchor">
<h4>Hybridization<a href="speciation-1.html#hybridization" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Hybridization can be a powerful motor for generating functional diversity early during a radiation. Research, especially on Lake Victoria and Malawi cichlids, has indicated that these lakes were likely colonized by one or a few closely related forms that adapted to different ecological niches. Upon initial diversification, hybridization between species then suddenly created a wealth of new phenotypes. In an environment with a lot of open niches, intermediate phenotypes that arise from hybridization may be well-adapted to exploit a resource that is underused. Furthermore, hybridization can also create novel phenotypes that are outside the range of both parental species, a phenomenon called transgressive segregation. Transgressive segregation is caused by the recombination of genes with non-additive genetic effects, and the resulting novel phenotypes may be able to exploit resources that are inaccessible to the parental forms.</p>
<p>Hybridization can consequently lead to a burst of functional diversification. As long as open niches are available, there may be little or no cost to hybridization, because competition is low. However, as niches fill up and competition intensifies, the situation changes, and with increasing costs of hybridization, reinforcement should facilitate the evolution of reproductive barriers that reduce interspecific matings. Over time, competitive exclusion leads to the disappearance of suboptimal forms that cannot compete with others. As a consequence, species diversity does not continuously increase with the age of an adaptive radiation. Rather, species diversity is highest in relatively young adaptive radiations, where novel forms are generated rapidly through hybridization, and before selection eliminates forms that do not fall within a unique niche. The non-constant rate of speciation explains why some young adaptive radiations (like Lake Victoria) have a lot more species than older adaptive radiations (Lake Tanganyika).</p>
</div>
<div id="microallopatry-and-lake-level-fluctuations" class="section level4 unnumbered hasAnchor">
<h4>Microallopatry and Lake Level Fluctuations<a href="speciation-1.html#microallopatry-and-lake-level-fluctuations" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Lakes are contiguous bodies of water, and as such they seem free of any geographic barriers. Even a small fish should, in theory, be able to swim anywhere it pleases. So, at first sight, speciation within each lake seems to be happening largely in sympatry. In practice, however, fish do not just swim anywhere they please, but have fairly strict habitat associations. For example, cichlid species that are associated with rocky reef habitats are philopatric, and even short stretches of sand between rocky reefs represent a significant physical barrier that these fish do not regularly cross (a phenomenon called microallopatry). Hence, these large lakes should not be viewed as contiguous, but as composed of distinct and physically isolated habitat patches. In many ways, the rocky reefs act like isolated islands in a sea of sand. Consequently, allopatric speciation processes can also contribute to species diversification.</p>
<p>Besides microallopatry, lake level fluctuations can impact speciation processes. Lake Tanganyika, for example, became subdivided into multiple distinct lakes when water levels fell in the past. Divergence in allopatry and completion of the speciation process upon secondary contact explains divergence patterns in multiple groups of cichlids. In addition, lake level fluctuations can drastically change the availability of different habitat types. Imagine a population of cichlids that occupies a rocky shore when the lake level is low (Figure <a href="speciation-1.html#fig:speciespump">11.22</a>A). As lake levels rise, some individuals of that species might be able to colonize new rocky reef habitats in shallower water (Figure <a href="speciation-1.html#fig:speciespump">11.22</a>B). Over time, the populations diverge in allopatry, for example, due to adaptation to local environmental conditions (Figure <a href="speciation-1.html#fig:speciespump">11.22</a>C). As lake levels drop and habitats in shallow areas of the lake disappear, species have to retreat to suitable habitats in deeper waters, bringing the divergent lineages into secondary contact (Figure <a href="speciation-1.html#fig:speciespump">11.22</a>D). In sympatry, competitive exclusion may cause some lineages to disappear, or reinforcement could strengthen reproductive isolation and push the speciation process forward. If lake levels rise again, the newly formed species once again disperse to novel habitats in shallow water (Figure <a href="speciation-1.html#fig:speciespump">11.22</a>E), where they diverge in isolation (Figure <a href="speciation-1.html#fig:speciespump">11.22</a>F). As the process repeats itself, more and more species are generated until all niches are occupied by corresponding specialists. This process is called a “species-pump” and illustrates how changes in environmental conditions, in conjunction with local adaptation, can generate species diversity on small spatial scales. Some rocky reefs in Lake Malawi, which harbor unique microendemics found nowhere else in the lake, were dry just over 100 years ago when lake levels were significantly lower than today—suggesting that processes associated with species pumps are ongoing today and rapid.</p>
<div class="figure"><span style="display:block;" id="fig:speciespump"></span>
<img src="images/speciespump.png" alt="Lake level fluctuations can drive speciation if they cause the repeated splitting and fusion of species (see text for details. Dark blue areas indicate deep areas of the lake, light blue areas shallow areas. Rocky reefs are shown in brown, and the colored circle represent different species  as they move between habitats " width="100%" />
<p class="caption">
Figure 11.22: Lake level fluctuations can drive speciation if they cause the repeated splitting and fusion of species (see text for details. Dark blue areas indicate deep areas of the lake, light blue areas shallow areas. Rocky reefs are shown in brown, and the colored circle represent different species as they move between habitats
</p>
</div>
</div>
<div id="sexual-selection-1" class="section level4 unnumbered hasAnchor">
<h4>Sexual Selection<a href="speciation-1.html#sexual-selection-1" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Last but not least, sexual selection has played an important role in the diversification of cichlid fishes. Sexual selection is generally strong in most cichlids from Africa’s Rift lakes, since females incubate their offspring in their mouths for extended periods of time, and males court in leks without contributing to reproduction beyond the provisioning of sperm. The effect of strong sexual selection is evidenced by the wealth of species that appear to be morphologically and ecologically similar, but exhibit differences in the coloration of courting males. Even subtle differences in coloration are associated with strong patterns of assortative mating. For example, in the <em>Maylandia zebra</em> species complex of Lake Malawi, some sympatric species differ only in the extent of blue vs. orange coloration in the dorsal fin (Blais et al. 2009). Females detect these subtle differences and have clear mating preferences for their own male type (Figure <a href="speciation-1.html#fig:cassmat">11.23</a>). More so, even allopatric forms that look nearly indistinguishable to the human eye will mate assortatively under laboratory conditions (Figure <a href="speciation-1.html#fig:cassmat">11.23</a>). This suggests that strong divergence in female preferences and male nuptial coloration, even between closely related species and populations, represent strong reproductive barriers keeping species apart.</p>
<div class="figure"><span style="display:block;" id="fig:cassmat"></span>
<img src="Primer2Evolution_files/figure-html/cassmat-1.png" alt="Mating trials with cichlids of the genus *Maylandia* indicated that both sympatric species with different color patterns and allopatric species with similar color patterns are reproductively isolated through female mate choice. [Data](data/11_cichlid-assmat.csv) from Blais et al. (2009)." width="100%" />
<p class="caption">
Figure 11.23: Mating trials with cichlids of the genus <em>Maylandia</em> indicated that both sympatric species with different color patterns and allopatric species with similar color patterns are reproductively isolated through female mate choice. <a href="data/11_cichlid-assmat.csv">Data</a> from Blais et al. (2009).
</p>
</div>
</div>
</div>
</div>
<div id="case-study-speciation" class="section level2 hasAnchor" number="11.6">
<h2><span class="header-section-number">11.6</span> Case Study: Speciation<a href="speciation-1.html#case-study-speciation" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>The <a href="exercises/BIOL520-ex10.zip">R exercise associated with this chapter</a> includes four case studies that further explore concepts associated with species and speciation:</p>
<ol style="list-style-type: decimal">
<li><p>In the first part of this exercise, you will visualize data about morphological, phylogenetic, and behavioral traits in two putative species of butterfly and use this information to evaluate how many species are actually represented.</p></li>
<li><p>In the second case study, you will analyze data from a classic experiment using <em>Drosophila</em> fruitflies. Researchers conducted mate choice trials between a large number of species with sympatric and allopatric distributions, which will allow you to make inferences about the evolution of pre-mating isolation.</p></li>
<li><p>The third case study focuses on an evolution experiment, also using <em>Drosophila</em>. Researchers bred a number of replicated lines under different environmental conditions for many generations and then analyzed patterns of reproductive isolation across lines. The results of this experiment will allow you to make inferences about how adaptation to different environmental conditions relates to the emergence of reproductive isolation.</p></li>
<li><p>Last but not least, you will explore a case study of ongoing speciation in extreme environments. Analyzing experimental and population genetic datasets, you will test whether trait divergence across populations is caused by evolutionary change, and how adaptation to extreme environments might impact reproductive isolation across small spatial scales.</p></li>
</ol>
</div>
<div id="practical-skills-interpreting-structure-plots" class="section level2 hasAnchor" number="11.7">
<h2><span class="header-section-number">11.7</span> Practical Skills: Interpreting Structure Plots<a href="speciation-1.html#practical-skills-interpreting-structure-plots" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>From a coding perspective, there are no new graph types you need to learn about for this exercise; you will be able to apply skills that you have already learned over the past chapters. However, you will have a chance to explore some population genetic data that are commonly used to investigate ongoing speciation with gene flow. Specifically, analyses of population structure use genetic data from multiple loci to test hypotheses about the most likely number of genetically distinct clusters in a set of samples. These analyses try to identify the systematic differences in allele frequencies that arise between subpopulations as a consequence of nonrandom mating. Results from analyses of population structure are typically visualized using stacked bar plots, where each bar corresponds to an individual, and bar segments are proportional to the probability of an individual belonging to a particular cluster (this is equivalent to the proportion of an individual’s genetic ancestry from one of the clusters). Sounds abstract? Let’s look at some concrete scenarios.</p>
<p>Suppose we found this really cool species of bird that looks slightly different across two habitats, and we wanted to know whether the birds in the different habitats constitute different populations with reduced gene flow between them (<em>i.e.</em>, we want to know whether there is population structure or not). So we go out, sample 20 birds from each habitat, sequence some genes, and we calculate the probability of each individual belonging to either of two hypothesized subgroups based on allele frequency differences. If there is no population differentiation, and all birds across both habitats represent one randomly mating population, the probability for each individual belonging to either one of the two clusters is 50 %—that is no better than chance! This is illustrated in Figure <a href="speciation-1.html#fig:admixture">11.24</a>A: statistically, we cannot tell apart individuals from the two habitat types based on their genetic makeup.</p>
<p>At the other end of the extreme, we might find very strong population structure, with two randomly mating populations but no interbreeding between them. In that case, we may be able to assign each individual to one of two clusters with 100 % probability. This scenario is illustrated in Figure <a href="speciation-1.html#fig:admixture">11.24</a>B; individuals from each habitat are distinct and clearly assignable to one of two clusters based on their multilocus genotype.</p>
<p>In many cases, the actual results are somewhere in between (Figure <a href="speciation-1.html#fig:admixture">11.24</a>C). In this case, there is significant population genetic differentiation, but there are also some individuals where the assignment probabilities are somewhere between 0 and 100 %. Such individuals provide evidence for recent gene flow between clusters through interbreeding. For example, individuals with an assignment probability of 50 % in this scenario are putative F1 hybrids between individuals from the two populations.</p>
<div class="figure"><span style="display:block;" id="fig:admixture"></span>
<img src="Primer2Evolution_files/figure-html/admixture-1.png" alt="A. Hypothetical structure plot with no genetic differentiation between populations. The assignment probability for each individual to either cluster 1 or cluster 2 is 50%, which is equal to random chance. B. Hypothetical structure plot with strong genetic differentiation between populations. The assignment probability for each individual to one of the two clusters is 100%, meaning that the individuals from each populations are genetically distinct. C. Hypothetical structure plot with strong genetic differentiation and evidence for admixture between the clusters. Individuals with intermediate assignment probabilities provide evidence for recent interbreeding between the clusters." width="100%" />
<p class="caption">
Figure 11.24: A. Hypothetical structure plot with no genetic differentiation between populations. The assignment probability for each individual to either cluster 1 or cluster 2 is 50%, which is equal to random chance. B. Hypothetical structure plot with strong genetic differentiation between populations. The assignment probability for each individual to one of the two clusters is 100%, meaning that the individuals from each populations are genetically distinct. C. Hypothetical structure plot with strong genetic differentiation and evidence for admixture between the clusters. Individuals with intermediate assignment probabilities provide evidence for recent interbreeding between the clusters.
</p>
</div>
<p>It’s important to note that structure analyses may reveal more clusters than just two (Figure <a href="speciation-1.html#fig:admixture2">11.25</a>). The number of clusters in a given sample also does not have to be determined <em>a priori</em>. Rather, analyses of genetic structure are run for different numbers of clusters, and the actual number of cluster can then be determined empirically.</p>
<div class="figure"><span style="display:block;" id="fig:admixture2"></span>
<img src="Primer2Evolution_files/figure-html/admixture2-1.png" alt=" Hypothetical structure plot with three clusters and evidence for some genetic differentiation." width="100%" />
<p class="caption">
Figure 11.25:  Hypothetical structure plot with three clusters and evidence for some genetic differentiation.
</p>
</div>
</div>
<div id="reflection-questions-10" class="section level2 hasAnchor" number="11.8">
<h2><span class="header-section-number">11.8</span> Reflection Questions<a href="speciation-1.html#reflection-questions-10" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<ol style="list-style-type: decimal">
<li><p>What does it mean when we say that species are “evolutionarily independent” or that they form the boundaries for gene flow?</p></li>
<li><p>What are the primary differences and commonalities of speciation by polyploidization, allopatric speciation, and ecological speciation?</p></li>
<li><p>How long do you think it takes for speciation to occur? Do you think the speed of speciation varies among different modes of speciation?</p></li>
<li><p>The closing of the Isthmus of Panama and the formation of the Central American land bridge had massive repercussions for biogeography. Can you contrast how this geological event may have impacted the evolution of terrestrial and marine organisms in the region?</p></li>
<li><p>Can you describe the diverse ways hybridization may impact the evolution of biodiversity?</p></li>
<li><p>What are the key ingredients for the occurrence of adaptive radiation?</p></li>
</ol>
</div>
<div id="references-11" class="section level2 hasAnchor" number="11.9">
<h2><span class="header-section-number">11.9</span> References<a href="speciation-1.html#references-11" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<ul>
<li><p>Blais J, M Plenderleith, C Rico, MI Taylor, O Seehausen, C van Oosterhout, GF Turner (2009). <a href="https://bmcecolevol.biomedcentral.com/articles/10.1186/1471-2148-9-53">Assortative mating among Lake Malawi cichlid fish populations is not simply predictable from male nuptial colour</a>. <em>BMC Evolutionary Biology</em> 9, 53.</p></li>
<li><p>Bolnick DI, LK Snowberg, C Patenia, WE Stutz, T Ingram, OL Lau (2009). <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1558-5646.2009.00699.x">Phenotype-dependent native habitat preference facilitates divergence between parapatric lake and stream stickleback</a>. <em>Evolution</em> 63, 2004–2016.</p></li>
<li><p>De Queiroz K (2007). <a href="https://academic.oup.com/sysbio/article/56/6/879/1653163">Species concepts and species delimitation</a>. <em>Systematic Biology</em> 56, 879–886.</p></li>
<li><p>Hillis DM (1981). <a href="https://www.jstor.org/stable/1444220?origin=crossref&amp;seq=1#metadata_info_tab_contents">Premating isolating mechanisms among three species of the <em>Rana pipiens</em> complex in Texas and southern Oklahoma</a>. <em>Copeia</em> 1981, 312.</p></li>
<li><p>Hodges SA, ML Arnold (1994). <a href="https://www.pnas.org/content/91/7/2493">Floral and ecological isolation between <em>Aquilegia formosa</em> and <em>Aquilegia pubescens</em></a>. <em>Proceedings of the National Academy of Sciences USA</em> 91, 2493–2496.</p></li>
<li><p>Jiggins CD, RE Naisbit, RL Coe, J Mallet (2001). <a href="https://www.nature.com/articles/35077075">Reproductive isolation caused by colour pattern mimicry</a>. <em>Nature</em> 411, 302–305.</p></li>
<li><p>Langerhans RB, ME Gifford, EO Joseph (2007). <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1558-5646.2007.00171.x">Ecological speciation in <em>Gambusia</em> fishes</a>. <em>Evolution</em> 61, 2056–2074.</p></li>
<li><p>Naisbit RE, CD Jiggins, J Mallet (2001). <a href="https://royalsocietypublishing.org/doi/10.1098/rspb.2001.1753">Disruptive sexual selection against hybrids contributes to speciation between Heliconius cydno and Heliconius melpomene</a>. <em>Proceedings of the Royal Society B</em> 268, 1849–1854.</p></li>
<li><p>Obbard DJ, J Maclennan, KW Kim, A Rambaut, PM O’Grady, FM Jiggins (2012). <a href="https://academic.oup.com/mbe/article/29/11/3459/1150622">Estimating divergence dates and substitution rates in the <em>Drosophila</em> phylogeny</a>. <em>Molecular Biology and Evolution</em> 29, 3459–3473.</p></li>
<li><p>Palumbi SR, EC Metz (1991). <a href="https://academic.oup.com/mbe/article/8/2/227/1134109">Strong reproductive isolation between closely related tropical sea urchins (genus <em>Echinometra</em>)</a>. <em>Molecular Biology and Evolution</em> 8, 227–239.</p></li>
<li><p>Podos J (2001). <a href="https://www.nature.com/articles/35051570">Correlated evolution of morphology and vocal signal structure in Darwin’s finches</a>. <em>Nature</em> 409, 185–188.</p></li>
<li><p>Powell DL, M García-Olazábal, M Keegan, P Reilly, K Du, AP Díaz-Loyo, … M Schumer (2020). <a href="https://www.science.org/lookup/doi/10.1126/science.aba5216">Natural hybridization reveals incompatible alleles that cause melanoma in swordtail fish</a>. <em>Science</em> 368, 731–736.</p></li>
<li><p>Riechert SE, RF Hall (2000). <a href="https://onlinelibrary.wiley.com/doi/abs/10.1046/j.1420-9101.2000.00176.x">Local population success in heterogeneous habitats: reciprocal transplant experiments completed on a desert spider</a>. <em>Journal of Evolutionary Biology</em> 13, 541–550.</p></li>
<li><p>Sætre GP, T Moum, S Bureš, M Král, M Adamjan, J Moreno (1997). <a href="https://www.nature.com/articles/42451">A sexually selected character displacement in flycatchers reinforces premating isolation</a>. <em>Nature</em> 387, 589–592.</p></li>
<li><p>Schluter D (1995). <a href="https://www.jstor.org/stable/1940633?seq=1#metadata_info_tab_contents">Adaptive radiation in sticklebacks: trade-offs in feeding performance and growth</a>. <em>Ecology</em> 76, 82–90.</p></li>
<li><p>Schluter D (2000). <em>The Ecology of Adaptive Radiation</em>. Oxford: Oxford University Press.</p></li>
<li><p>Seehausen O (2006). <a href="https://royalsocietypublishing.org/doi/10.1098/rspb.2006.3539">African cichlid fish: a model system in adaptive radiation research</a>. <em>Proceedings of the Royal Society B</em> 273, 1987–1998.</p></li>
<li><p>Seehausen O, Y Terai, IS Magalhaes, KL Carleton, HDJ Mrosso, R Miyagi R, … Okada N (2008). <a href="https://www.nature.com/articles/nature07285">Speciation through sensory drive in cichlid fish</a>. <em>Nature</em> 455, 620–627.</p></li>
<li><p>Villa SM, JC Altuna, JS Ruff, AB Beach, LI Mulvey, EJ Poole … DH Clayton (2019). <a href="https://www.pnas.org/content/116/27/13440">Rapid experimental evolution of reproductive isolation from a single natural population</a>. <em>Proceedings of the National Academy of Sciences USA</em> 116, 13440–13445.</p></li>
</ul>

</div>
</div>



            </section>

          </div>
        </div>
      </div>
<a href="social-behavior-and-sexual-selection.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="evolutionary-medicine-i-aging-and-diseases-of-civilization.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": "nullnull",
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"toolbar": {
"position": "fixed"
},
"info": false
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
